from collections import OrderedDict

# output from elife04490.xml
expected = [
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.001"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.001"),
            (
                "content",
                "Plants are at the trophic base of terrestrial ecosystems, and the diversity of plant species in an ecosystem is a principle determinant of community structure. This may arise from diverse functional traits among species. In fact, genetic diversity within species can have similarly large effects. However, studies of intraspecific genetic diversity have used genotypes varying in several complex traits, obscuring the specific phenotypic variation responsible for community-level effects. Using lines of the wild tobacco Nicotiana attenuata genetically altered in specific well-characterized defense traits and planted into experimental populations in their native habitat, we investigated community-level effects of trait diversity in populations of otherwise isogenic plants. We conclude that the frequency of defense traits in a population can determine the outcomes of these traits for individuals. Furthermore, our results suggest that some ecosystem-level services afforded by genetically diverse plant populations could be recaptured in intensive monocultures engineered to be functionally diverse. DOI: http://dx.doi.org/10.7554/eLife.04490.001",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "abstract"),
            ("position", 1),
            ("ordinal", 1),
            ("sibling_ordinal", 1),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.002"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.002"),
            ("title", "eLife digest"),
            ("full_title", "eLife digest"),
            (
                "content",
                "Plants are at the base of many food webs. This means that the different traits and characteristics of the plant species in an ecosystem can have a large impact on the animals and other organisms that live there. Individuals within the same plant species often differ in multiple genes. This ‘genetic diversity’ can affect the populations of other organisms in their ecosystem, for example, by altering which species are present, and the number of individuals. However, previous studies that investigated plant genetic diversity in ecosystems used plants that varied in multiple, usually unknown, genetic traits, which made it difficult to identify specific genetic traits in plants that can influence the whole ecosystem. One way that plants affect their ecosystems involves how they defend themselves against the herbivores that try to eat them. For example, plants can use sharp spines or harmful chemicals to deter herbivores, or attract predators that will attack the herbivores. Here, Schuman et al. carried out a 2-year field study using wild tobacco plants that had been genetically altered to employ different defensive strategies. This was achieved by altering the expression of three genes in the plants in specific combinations. Two of the genes, called LOX2 and LOX3, are required to make most of the chemicals that tobacco plants use to defend themselves against herbivores. The third gene, called TPS10, which comes from the crop plant maize, gives plants the ability to release a fragrance that attracts natural predators of their herbivores. Except for these specific alterations, the plants were otherwise genetically identical. These plants were then grown either alone, or in groups of five plants, which reflects the normal size of groups of wild tobacco growing in its natural environment. The groups contained mixtures of plants with different gene alterations. Schuman et al. found that the expression levels of these genes in individual plants could determine how well the whole group fared in several different measures of plant health and defense. For example, plants lacking LOX2 and LOX3 usually appeared to be less healthy and they were less likely to survive long enough to reproduce because they were less able to defend themselves against herbivores. However, if these plants were grown in a group with one plant that expressed TPS10, all of the plants in the group looked healthier and were more likely to survive long enough to reproduce. Many crops are grown in large fields containing individual plants that are largely genetically identical, which makes them more vulnerable to disease. Schuman et al.'s findings suggest that some of the community-level protective effects provided by diversity in wild populations of plants could be introduced into crop fields by altering the expression of a few specific genes in some individuals. DOI: http://dx.doi.org/10.7554/eLife.04490.002",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "abstract"),
            ("position", 2),
            ("ordinal", 2),
            ("sibling_ordinal", 2),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.003"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.003"),
            ("id", "fig1"),
            (
                "title",
                "The lox2/3 and TPS10 transgenic constructs, alone and in combination (lox2/3xTPS10), independently alter herbivore-induced plant defenses and sesquiterpene HIPVs.",
            ),
            (
                "full_title",
                "The <italic>lox2/3</italic> and <italic>TPS10</italic> transgenic constructs, alone and in combination (<italic>lox2/3xTPS10</italic>), independently alter herbivore-induced plant defenses and sesquiterpene HIPVs.",
            ),
            ("label", "Figure 1."),
            ("full_label", "Figure 1."),
            (
                "caption",
                "(A) The endogenous lipoxygenase genes LOX2 and LOX3 and the Z. mays sesquiterpene synthase gene TPS10 (TPS10) were manipulated in transgenic lines of N. attenuata in order to uncouple the production of two HIPVs—the sesquiterpenes (E)-α-bergamotene (TAB) and (E)-β-farnesene (TBF)—from jasmonate-mediated direct defenses and other volatiles. LOX2 and LOX3 provide fatty acid hydroperoxides from 18:2 and 18:3 fatty acids (18:2/18:3-OOH) for the synthesis of green leaf volatiles (GLVs) or jasmonic acid (JA) and the jasmonate-derived hormones, respectively. GLVs are released in response to herbivore damage, and jasmonates regulate the production of most anti-herbivore defenses, including trypsin protease inhibitors (TPIs), and other HIPVs. LOX2 and LOX3 were transgenically silenced using an inverted repeat (ir) construct specifically silencing both (lox2/3). Jasmonate-regulated volatiles in N. attenuata are mostly sesquiterpenes, of which the most prevalent is TAB, and include trace amounts of the biosynthetically related sesquiterpene TBF. The Z. mays sesquiterpene synthase TPS10 (Köllner et al., 2009), which produces TAB and TBF as its main products, was ectopically over-expressed in N. attenuata plants under control of a 35S promoter (TPS10) to uncouple the emission of these volatiles from endogenous jasmonate signaling. (B and C) Plants with lox2/3 and TPS10 constructs, and a cross with both constructs having the transgenes in a hemizygous state (lox2/3xTPS10) had the expected phenotypes both in glasshouse and field experiments. For each compound or group of compounds, the mean WT value was set to 1 (indicated by dashed lines), and all values were divided by the WT mean resulting in mean fold changes ±SEM. A complete description of TPS10 plants including the demonstration of a single transgene insertion and WT levels of non-target metabolites is provided in Schuman et al. (2014), and additional information for lox2/3, TPS10, and lox2/3xTPS10 plants (single transgene insertion for lox2/3, accumulation of target gene transcripts) is given in Appendix 1. (B) Glasshouse-grown plants were treated with wounding and M. sexta OS, and JA, GLVs, TAB and TBF were analyzed at the time of peak accumulation; n = 4. For lox2/3 and lox2/3xTPS10, GLVs were not quantifiable (NQ) by GC–MS due to inconsistently detected or no detected signals. ***p < 0.001 in Tukey HSD tests following a significant one-way ANOVA (p < 0.001) of WT, lox2/3, and lox2/3xTPS10; # plants with the TPS10 construct emit significantly more TAB + TBF (Holm-Bonferroni-corrected p < 0.01 in a Wilcoxon rank sum test of WT and lox2/3 vs lox2/3xTPS10 and TPS10); ns, not significant. TBF was not detectable in lox2/3 or WT samples. For absolute amounts of JA and other phytohormones, GLVs, TAB and TBF measured in glasshouse samples, see Figure 1—source data 1, 2. (C) TPI activity, total detectable GLVs, and total TAB and TBF measured in frozen leaf samples harvested from field-grown plants at the end of experimental season two (June 28th), after plants had accumulated damage from naturally occurring herbivores. TPIs (n = 11–24) were slightly elevated in field samples of TPS10 plants, but TPI activity did not differ from WT plants in glasshouse samples from two independent TPS10 lines including the line used for this study (Schuman et al., 2014). Total GLVs, TAB, and TBF were quantified by GC–MS in hexane extracts from the same tissue samples used to measure TPI activity, n = 9–22 (a few samples had insufficient tissue for the analysis). Neither TAB nor TBF could be detected in lox2/3 samples (NQ); TBF was also not detectable in WT samples. *Corrected p < 0.05, ***corrected p < 0.001 in pairwise Wilcoxon rank sum tests following significant (corrected p < 0.05) Kruskal–Wallis tests across all genotypes for each category; p-values were corrected for multiple testing using the Holm-Bonferroni method; ns, not significant. For absolute amounts of TPIs, GLVs, TAB and TBF, and non-target volatiles measured in field-collected tissue samples, see Figure 1—source data 3; emission of GLVs, TAB, TBF, and non-target volatiles from field-grown plants is given in Appendix 2.",
            ),
            (
                "full_caption",
                '(<bold>A</bold>) The endogenous lipoxygenase genes <italic>LOX2</italic> and <italic>LOX3</italic> and the <italic>Z. mays</italic> sesquiterpene synthase gene <italic>TPS10</italic> (<italic>TPS10</italic>) were manipulated in transgenic lines of <italic>N. attenuata</italic> in order to uncouple the production of two HIPVs—the sesquiterpenes (<italic>E</italic>)-α-bergamotene (TAB) and (<italic>E</italic>)-β-farnesene (TBF)—from jasmonate-mediated direct defenses and other volatiles. LOX2 and LOX3 provide fatty acid hydroperoxides from 18:2 and 18:3 fatty acids (18:2/18:3-OOH) for the synthesis of green leaf volatiles (GLVs) or jasmonic acid (JA) and the jasmonate-derived hormones, respectively. GLVs are released in response to herbivore damage, and jasmonates regulate the production of most anti-herbivore defenses, including trypsin protease inhibitors (TPIs), and other HIPVs. LOX2 and LOX3 were transgenically silenced using an inverted repeat (ir) construct specifically silencing both (<italic>lox2/3</italic>). Jasmonate-regulated volatiles in <italic>N. attenuata</italic> are mostly sesquiterpenes, of which the most prevalent is TAB, and include trace amounts of the biosynthetically related sesquiterpene TBF. The <italic>Z. mays</italic> sesquiterpene synthase <italic>TPS10</italic> (Köllner et al., 2009), which produces TAB and TBF as its main products, was ectopically over-expressed in <italic>N. attenuata</italic> plants under control of a 35S promoter (<italic>TPS10</italic>) to uncouple the emission of these volatiles from endogenous jasmonate signaling. (<bold>B</bold> and <bold>C</bold>) Plants with <italic>lox2/3</italic> and <italic>TPS10</italic> constructs, and a cross with both constructs having the transgenes in a hemizygous state (<italic>lox2/3xTPS10</italic>) had the expected phenotypes both in glasshouse and field experiments. For each compound or group of compounds, the mean WT value was set to 1 (indicated by dashed lines), and all values were divided by the WT mean resulting in mean fold changes ±SEM. A complete description of <italic>TPS10</italic> plants including the demonstration of a single transgene insertion and WT levels of non-target metabolites is provided in <xref ref-type="bibr" rid="bib71">Schuman et al. (2014)</xref>, and additional information for <italic>lox2/3</italic>, <italic>TPS10</italic>, and <italic>lox2/3xTPS10</italic> plants (single transgene insertion for <italic>lox2/3</italic>, accumulation of target gene transcripts) is given in <xref ref-type="app" rid="app1">Appendix 1</xref>. (<bold>B</bold>) Glasshouse-grown plants were treated with wounding and <italic>M</italic>. <italic>sexta</italic> OS, and JA, GLVs, TAB and TBF were analyzed at the time of peak accumulation; n = 4. For <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic>, GLVs were not quantifiable (NQ) by GC–MS due to inconsistently detected or no detected signals. ***p < 0.001 in Tukey HSD tests following a significant one-way ANOVA (p < 0.001) of WT, <italic>lox2/3</italic>, and <italic>lox2/3xTPS10</italic>; # plants with the <italic>TPS10</italic> construct emit significantly more TAB + TBF (Holm-Bonferroni-corrected p < 0.01 in a Wilcoxon rank sum test of WT and <italic>lox2/3</italic> vs <italic>lox2/3xTPS10</italic> and <italic>TPS10</italic>); ns, not significant. TBF was not detectable in <italic>lox2/3</italic> or WT samples. For absolute amounts of JA and other phytohormones, GLVs, TAB and TBF measured in glasshouse samples, see <xref ref-type="supplementary-material" rid="SD1-data SD2-data">Figure 1—source data 1, 2</xref>. (<bold>C</bold>) TPI activity, total detectable GLVs, and total TAB and TBF measured in frozen leaf samples harvested from field-grown plants at the end of experimental season two (June 28th), after plants had accumulated damage from naturally occurring herbivores. TPIs (n = 11–24) were slightly elevated in field samples of <italic>TPS10</italic> plants, but TPI activity did not differ from WT plants in glasshouse samples from two independent <italic>TPS10</italic> lines including the line used for this study (<xref ref-type="bibr" rid="bib71">Schuman et al., 2014</xref>). Total GLVs, TAB, and TBF were quantified by GC–MS in hexane extracts from the same tissue samples used to measure TPI activity, n = 9–22 (a few samples had insufficient tissue for the analysis). Neither TAB nor TBF could be detected in <italic>lox2/3</italic> samples (NQ); TBF was also not detectable in WT samples. *Corrected p < 0.05, ***corrected p < 0.001 in pairwise Wilcoxon rank sum tests following significant (corrected p < 0.05) Kruskal–Wallis tests across all genotypes for each category; p<italic>-</italic>values were corrected for multiple testing using the Holm-Bonferroni method; ns, not significant. For absolute amounts of TPIs, GLVs, TAB and TBF, and non-target volatiles measured in field-collected tissue samples, see <xref ref-type="supplementary-material" rid="SD3-data">Figure 1—source data 3</xref>; emission of GLVs, TAB, TBF, and non-target volatiles from field-grown plants is given in <xref ref-type="app" rid="app2">Appendix 2</xref>.',
            ),
            (
                "content",
                "(A) The endogenous lipoxygenase genes LOX2 and LOX3 and the Z. mays sesquiterpene synthase gene TPS10 (TPS10) were manipulated in transgenic lines of N. attenuata in order to uncouple the production of two HIPVs—the sesquiterpenes (E)-α-bergamotene (TAB) and (E)-β-farnesene (TBF)—from jasmonate-mediated direct defenses and other volatiles. LOX2 and LOX3 provide fatty acid hydroperoxides from 18:2 and 18:3 fatty acids (18:2/18:3-OOH) for the synthesis of green leaf volatiles (GLVs) or jasmonic acid (JA) and the jasmonate-derived hormones, respectively. GLVs are released in response to herbivore damage, and jasmonates regulate the production of most anti-herbivore defenses, including trypsin protease inhibitors (TPIs), and other HIPVs. LOX2 and LOX3 were transgenically silenced using an inverted repeat (ir) construct specifically silencing both (lox2/3). Jasmonate-regulated volatiles in N. attenuata are mostly sesquiterpenes, of which the most prevalent is TAB, and include trace amounts of the biosynthetically related sesquiterpene TBF. The Z. mays sesquiterpene synthase TPS10 (Köllner et al., 2009), which produces TAB and TBF as its main products, was ectopically over-expressed in N. attenuata plants under control of a 35S promoter (TPS10) to uncouple the emission of these volatiles from endogenous jasmonate signaling. (B and C) Plants with lox2/3 and TPS10 constructs, and a cross with both constructs having the transgenes in a hemizygous state (lox2/3xTPS10) had the expected phenotypes both in glasshouse and field experiments. For each compound or group of compounds, the mean WT value was set to 1 (indicated by dashed lines), and all values were divided by the WT mean resulting in mean fold changes ±SEM. A complete description of TPS10 plants including the demonstration of a single transgene insertion and WT levels of non-target metabolites is provided in Schuman et al. (2014), and additional information for lox2/3, TPS10, and lox2/3xTPS10 plants (single transgene insertion for lox2/3, accumulation of target gene transcripts) is given in Appendix 1. (B) Glasshouse-grown plants were treated with wounding and M. sexta OS, and JA, GLVs, TAB and TBF were analyzed at the time of peak accumulation; n = 4. For lox2/3 and lox2/3xTPS10, GLVs were not quantifiable (NQ) by GC–MS due to inconsistently detected or no detected signals. ***p < 0.001 in Tukey HSD tests following a significant one-way ANOVA (p < 0.001) of WT, lox2/3, and lox2/3xTPS10; # plants with the TPS10 construct emit significantly more TAB + TBF (Holm-Bonferroni-corrected p < 0.01 in a Wilcoxon rank sum test of WT and lox2/3 vs lox2/3xTPS10 and TPS10); ns, not significant. TBF was not detectable in lox2/3 or WT samples. For absolute amounts of JA and other phytohormones, GLVs, TAB and TBF measured in glasshouse samples, see Figure 1—source data 1, 2. (C) TPI activity, total detectable GLVs, and total TAB and TBF measured in frozen leaf samples harvested from field-grown plants at the end of experimental season two (June 28th), after plants had accumulated damage from naturally occurring herbivores. TPIs (n = 11–24) were slightly elevated in field samples of TPS10 plants, but TPI activity did not differ from WT plants in glasshouse samples from two independent TPS10 lines including the line used for this study (Schuman et al., 2014). Total GLVs, TAB, and TBF were quantified by GC–MS in hexane extracts from the same tissue samples used to measure TPI activity, n = 9–22 (a few samples had insufficient tissue for the analysis). Neither TAB nor TBF could be detected in lox2/3 samples (NQ); TBF was also not detectable in WT samples. *Corrected p < 0.05, ***corrected p < 0.001 in pairwise Wilcoxon rank sum tests following significant (corrected p < 0.05) Kruskal–Wallis tests across all genotypes for each category; p-values were corrected for multiple testing using the Holm-Bonferroni method; ns, not significant. For absolute amounts of TPIs, GLVs, TAB and TBF, and non-target volatiles measured in field-collected tissue samples, see Figure 1—source data 3; emission of GLVs, TAB, TBF, and non-target volatiles from field-grown plants is given in Appendix 2. DOI: http://dx.doi.org/10.7554/eLife.04490.003 10.7554/eLife.04490.004Figure 1—source data 1.Absolute values for JA and GLVs shown as fold-changes in Figure 1B in lox2/3 and lox2/3xTPS10 plants, as well as JA-Ile, individual GLVs, and leaf areas for headspace collection (mean ± SEM); n = 4.Data are from a single glasshouse experiment. Values for TPS10 (line 10–3) are reported in Schuman et al. (2014).DOI: http://dx.doi.org/10.7554/eLife.04490.004 Data are from a single glasshouse experiment. Values for TPS10 (line 10–3) are reported in Schuman et al. (2014). DOI: http://dx.doi.org/10.7554/eLife.04490.004 10.7554/eLife.04490.005Figure 1—source data 2.Absolute values for TAB and TBF shown as fold-changes in Figure 1B, and leaf areas for headspace collection (mean ± SEM); n = 4.Data are from a single glasshouse experiment.DOI: http://dx.doi.org/10.7554/eLife.04490.005 Data are from a single glasshouse experiment. DOI: http://dx.doi.org/10.7554/eLife.04490.005 10.7554/eLife.04490.006Figure 1—source data 3.Absolute values for analytes shown as fold-changes in Figure 1C (mean ± SEM).Data are from tissue samples of several pooled leaves per plant from field-grown plants, taken at the end of experimental season two (June 28th).DOI: http://dx.doi.org/10.7554/eLife.04490.006 Data are from tissue samples of several pooled leaves per plant from field-grown plants, taken at the end of experimental season two (June 28th). DOI: http://dx.doi.org/10.7554/eLife.04490.006",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 3),
            ("ordinal", 1),
            ("sibling_ordinal", 1),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.004"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.004"),
            ("id", "SD1-data"),
            (
                "title",
                "Absolute values for JA and GLVs shown as fold-changes in Figure 1B in lox2/3 and lox2/3xTPS10 plants, as well as JA-Ile, individual GLVs, and leaf areas for headspace collection (mean ± SEM); n = 4.",
            ),
            (
                "full_title",
                'Absolute values for JA and GLVs shown as fold-changes in <xref ref-type="fig" rid="fig1">Figure 1B</xref> in <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> plants, as well as JA-Ile, individual GLVs, and leaf areas for headspace collection (mean ± SEM); n = 4.',
            ),
            ("label", "Figure 1—source data 1."),
            ("full_label", "Figure 1—source data 1."),
            (
                "caption",
                "Data are from a single glasshouse experiment. Values for TPS10 (line 10–3) are reported in Schuman et al. (2014).",
            ),
            (
                "full_caption",
                'Data are from a single glasshouse experiment. Values for <italic>TPS10</italic> (line 10–3) are reported in <xref ref-type="bibr" rid="bib71">Schuman et al. (2014)</xref>.',
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 1),
            ("parent_sibling_ordinal", 1),
            ("parent_asset", None),
            (
                "content",
                "Data are from a single glasshouse experiment. Values for TPS10 (line 10–3) are reported in Schuman et al. (2014). DOI: http://dx.doi.org/10.7554/eLife.04490.004",
            ),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 4),
            ("ordinal", 1),
            ("sibling_ordinal", 1),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.005"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.005"),
            ("id", "SD2-data"),
            (
                "title",
                "Absolute values for TAB and TBF shown as fold-changes in Figure 1B, and leaf areas for headspace collection (mean ± SEM); n = 4.",
            ),
            (
                "full_title",
                'Absolute values for TAB and TBF shown as fold-changes in <xref ref-type="fig" rid="fig1">Figure 1B</xref>, and leaf areas for headspace collection (mean ± SEM); n = 4.',
            ),
            ("label", "Figure 1—source data 2."),
            ("full_label", "Figure 1—source data 2."),
            ("caption", "Data are from a single glasshouse experiment."),
            ("full_caption", "Data are from a single glasshouse experiment."),
            ("parent_type", "fig"),
            ("parent_ordinal", 1),
            ("parent_sibling_ordinal", 1),
            ("parent_asset", None),
            (
                "content",
                "Data are from a single glasshouse experiment. DOI: http://dx.doi.org/10.7554/eLife.04490.005",
            ),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 5),
            ("ordinal", 2),
            ("sibling_ordinal", 2),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.006"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.006"),
            ("id", "SD3-data"),
            (
                "title",
                "Absolute values for analytes shown as fold-changes in Figure 1C (mean ± SEM).",
            ),
            (
                "full_title",
                'Absolute values for analytes shown as fold-changes in <xref ref-type="fig" rid="fig1">Figure 1C</xref> (mean ± SEM).',
            ),
            ("label", "Figure 1—source data 3."),
            ("full_label", "Figure 1—source data 3."),
            (
                "caption",
                "Data are from tissue samples of several pooled leaves per plant from field-grown plants, taken at the end of experimental season two (June 28th).",
            ),
            (
                "full_caption",
                "Data are from tissue samples of several pooled leaves per plant from field-grown plants, taken at the end of experimental season two (June 28th).",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 1),
            ("parent_sibling_ordinal", 1),
            ("parent_asset", None),
            (
                "content",
                "Data are from tissue samples of several pooled leaves per plant from field-grown plants, taken at the end of experimental season two (June 28th). DOI: http://dx.doi.org/10.7554/eLife.04490.006",
            ),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 6),
            ("ordinal", 3),
            ("sibling_ordinal", 3),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.007"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.007"),
            ("id", "fig2"),
            (
                "title",
                "Field experiments designed to measure genotype-by-population effects of TPS10 volatiles in well-defended (WT, TPS10) or poorly defended (lox2/3, lox2/3xTPS10) plants.",
            ),
            (
                "full_title",
                "Field experiments designed to measure genotype-by-population effects of TPS10 volatiles in well-defended (WT, <italic>TPS10</italic>) or poorly defended (<italic>lox2/3</italic>, <italic>lox2/3xTPS10</italic>) plants.",
            ),
            ("label", "Figure 2."),
            ("full_label", "Figure 2."),
            (
                "caption",
                "Two similar experiments were conducted in two consecutive field seasons. (A) Each genotype was planted as individuals and in five-plant populations (see Figure 2—figure supplement 1). Each individual or population was planted ca. 1.5 m from the nearest neighboring individual/population, measured perimeter-to-perimeter. Populations consisted of five plants arranged as shown in a 0.4 × 0.4 m square. (B) Individual and population types: monocultures and, additionally, mixed cultures were planted in which a TPS10 plant (TPS10 or lox2/3xTPS10) was surrounded by four plants of the same level of jasmonate-mediated defense and GLVs (WT or lox2/3). Replicates (n = 12 in season one, 15 in season two of each individual or population type) were arranged in a blocked design (see Figure 2—figure supplement 2): blocks consisting of one replicate of each type (all six randomly-ordered populations followed by all four randomly-ordered individuals) were staggered such that consecutive blocks were not aligned, and thus populations and individuals were also interspersed. Random order was modified only when necessary to ensure that no two replicate individuals/populations were placed next to each other vertically, horizontally, or diagonally. This planting design reflects typical distributions in native populations of Nicotiana attenuata (see Figure 2—figure supplement 3), particularly before 1995, after which an invasive brome grass fueled a dramatic increase in fire sizes and consequently N. attenuata population sizes.",
            ),
            (
                "full_caption",
                'Two similar experiments were conducted in two consecutive field seasons. (<bold>A</bold>) Each genotype was planted as individuals and in five-plant populations (see <xref ref-type="fig" rid="fig2s1">Figure 2—figure supplement 1</xref>). Each individual or population was planted ca. 1.5 m from the nearest neighboring individual/population, measured perimeter-to-perimeter. Populations consisted of five plants arranged as shown in a 0.4 × 0.4 m square. (<bold>B</bold>) Individual and population types: monocultures and, additionally, mixed cultures were planted in which a <italic>TPS10</italic> plant (<italic>TPS10</italic> or <italic>lox2/3xTPS10</italic>) was surrounded by four plants of the same level of jasmonate-mediated defense and GLVs (WT or <italic>lox2/3</italic>). Replicates (n = 12 in season one, 15 in season two of each individual or population type) were arranged in a blocked design (see <xref ref-type="fig" rid="fig2s2">Figure 2—figure supplement 2</xref>): blocks consisting of one replicate of each type (all six randomly-ordered populations followed by all four randomly-ordered individuals) were staggered such that consecutive blocks were not aligned, and thus populations and individuals were also interspersed. Random order was modified only when necessary to ensure that no two replicate individuals/populations were placed next to each other vertically, horizontally, or diagonally. This planting design reflects typical distributions in native populations of <italic>Nicotiana attenuata</italic> (see <xref ref-type="fig" rid="fig2s3">Figure 2—figure supplement 3</xref>), particularly before 1995, after which an invasive brome grass fueled a dramatic increase in fire sizes and consequently <italic>N. attenuata</italic> population sizes.',
            ),
            (
                "content",
                "Two similar experiments were conducted in two consecutive field seasons. (A) Each genotype was planted as individuals and in five-plant populations (see Figure 2—figure supplement 1). Each individual or population was planted ca. 1.5 m from the nearest neighboring individual/population, measured perimeter-to-perimeter. Populations consisted of five plants arranged as shown in a 0.4 × 0.4 m square. (B) Individual and population types: monocultures and, additionally, mixed cultures were planted in which a TPS10 plant (TPS10 or lox2/3xTPS10) was surrounded by four plants of the same level of jasmonate-mediated defense and GLVs (WT or lox2/3). Replicates (n = 12 in season one, 15 in season two of each individual or population type) were arranged in a blocked design (see Figure 2—figure supplement 2): blocks consisting of one replicate of each type (all six randomly-ordered populations followed by all four randomly-ordered individuals) were staggered such that consecutive blocks were not aligned, and thus populations and individuals were also interspersed. Random order was modified only when necessary to ensure that no two replicate individuals/populations were placed next to each other vertically, horizontally, or diagonally. This planting design reflects typical distributions in native populations of Nicotiana attenuata (see Figure 2—figure supplement 3), particularly before 1995, after which an invasive brome grass fueled a dramatic increase in fire sizes and consequently N. attenuata population sizes. DOI: http://dx.doi.org/10.7554/eLife.04490.007",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 7),
            ("ordinal", 2),
            ("sibling_ordinal", 2),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.008"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.008"),
            ("id", "fig2s1"),
            ("title", "Photographs of the field experiment in season one."),
            ("full_title", "Photographs of the field experiment in season one."),
            ("label", "Figure 2—figure supplement 1."),
            ("full_label", "Figure 2—figure supplement 1."),
            (
                "caption",
                "(A) Photograph of a closed-headspace volatile trapping showing ca. 25% of the experiment in May of season one. (B) Photograph showing a similar portion of the experiment in June of season one.",
            ),
            (
                "full_caption",
                "(<bold>A</bold>) Photograph of a closed-headspace volatile trapping showing ca. 25% of the experiment in May of season one. (<bold>B</bold>) Photograph showing a similar portion of the experiment in June of season one.",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 2),
            ("parent_sibling_ordinal", 2),
            ("parent_asset", None),
            (
                "content",
                "(A) Photograph of a closed-headspace volatile trapping showing ca. 25% of the experiment in May of season one. (B) Photograph showing a similar portion of the experiment in June of season one. DOI: http://dx.doi.org/10.7554/eLife.04490.008",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 8),
            ("ordinal", 3),
            ("sibling_ordinal", 2),
            ("asset", "figsupp"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.009"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.009"),
            ("id", "fig2s2"),
            ("title", "Layout in experimental season two, and color codes."),
            ("full_title", "Layout in experimental season two, and color codes."),
            ("label", "Figure 2—figure supplement 2."),
            ("full_label", "Figure 2—figure supplement 2."),
            ("caption", "R, row of earth between two irrigation lines; not to scale."),
            (
                "full_caption",
                "R, row of earth between two irrigation lines; not to scale.",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 2),
            ("parent_sibling_ordinal", 2),
            ("parent_asset", None),
            (
                "content",
                "R, row of earth between two irrigation lines; not to scale. DOI: http://dx.doi.org/10.7554/eLife.04490.009",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 9),
            ("ordinal", 4),
            ("sibling_ordinal", 3),
            ("asset", "figsupp"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.010"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.010"),
            ("id", "fig2s3"),
            (
                "title",
                "Example of plant distribution in a native N. attenuata population, photographed in 2004.",
            ),
            (
                "full_title",
                "Example of plant distribution in a native <italic>N. attenuata</italic> population, photographed in 2004.",
            ),
            ("label", "Figure 2—figure supplement 3."),
            ("full_label", "Figure 2—figure supplement 3."),
            (
                "caption",
                "Reprinted with permission from Danny Kessler, Copyright 2004. All rights reserved.",
            ),
            (
                "full_caption",
                "Reprinted with permission from Danny Kessler, Copyright 2004. All rights reserved.",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 2),
            ("parent_sibling_ordinal", 2),
            ("parent_asset", None),
            (
                "content",
                "Reprinted with permission from Danny Kessler, Copyright 2004. All rights reserved. DOI: http://dx.doi.org/10.7554/eLife.04490.010",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 10),
            ("ordinal", 5),
            ("sibling_ordinal", 4),
            ("asset", "figsupp"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.011"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.011"),
            ("id", "fig3"),
            (
                "title",
                "The TPS10 product TAB can be detected in the open headspace around lox2/3 plants in mixed cultures containing a lox2/3xTPS10 plant.",
            ),
            (
                "full_title",
                "The TPS10 product TAB can be detected in the open headspace around <italic>lox2/3</italic> plants in mixed cultures containing a <italic>lox2/3xTPS10</italic> plant.",
            ),
            ("label", "Figure 3."),
            ("full_label", "Figure 3."),
            (
                "caption",
                "(A) An ‘open headspace’ trapping was conducted during season two (May 16th) with rosette-stage individuals and edge plants as shown, using activated charcoal filters shielded from ozone and UV by MnO2-coated copper ozone scrubbers (black strips in Teflon tubes pointed at plant) and aluminum foil, respectively. The headspace was sampled during the day, when sesquiterpenes are most abundant (see Appendix 2). Eluents from all four filters were combined and analyzed using highly sensitive GCxGC-ToF analysis; n = 4 plants. (B) The TPS10 product trans-α-bergamotene (TAB) could be detected in the open headspace of two of four lox2/3 plants at the edges of mixed cultures containing a lox2/3xTPS10 plant, but not in lox2/3 individuals or lox2/3 plants in monocultures. Peak areas were normalized to the internal standard (IS) peak. Arrows indicate the plant that was sampled.",
            ),
            (
                "full_caption",
                '(<bold>A</bold>) An ‘open headspace’ trapping was conducted during season two (May 16th) with rosette-stage individuals and edge plants as shown, using activated charcoal filters shielded from ozone and UV by MnO<sub>2</sub>-coated copper ozone scrubbers (black strips in Teflon tubes pointed at plant) and aluminum foil, respectively. The headspace was sampled during the day, when sesquiterpenes are most abundant (see <xref ref-type="app" rid="app2">Appendix 2</xref>). Eluents from all four filters were combined and analyzed using highly sensitive GCxGC-ToF analysis; n = 4 plants. (<bold>B</bold>) The <italic>TPS10</italic> product <italic>trans</italic>-α-bergamotene (TAB) could be detected in the open headspace of two of four <italic>lox2/3</italic> plants at the edges of mixed cultures containing a <italic>lox2/3xTPS10</italic> plant, but not in <italic>lox2/3</italic> individuals or <italic>lox2/3</italic> plants in monocultures. Peak areas were normalized to the internal standard (IS) peak. Arrows indicate the plant that was sampled.',
            ),
            (
                "content",
                "(A) An ‘open headspace’ trapping was conducted during season two (May 16th) with rosette-stage individuals and edge plants as shown, using activated charcoal filters shielded from ozone and UV by MnO2-coated copper ozone scrubbers (black strips in Teflon tubes pointed at plant) and aluminum foil, respectively. The headspace was sampled during the day, when sesquiterpenes are most abundant (see Appendix 2). Eluents from all four filters were combined and analyzed using highly sensitive GCxGC-ToF analysis; n = 4 plants. (B) The TPS10 product trans-α-bergamotene (TAB) could be detected in the open headspace of two of four lox2/3 plants at the edges of mixed cultures containing a lox2/3xTPS10 plant, but not in lox2/3 individuals or lox2/3 plants in monocultures. Peak areas were normalized to the internal standard (IS) peak. Arrows indicate the plant that was sampled. DOI: http://dx.doi.org/10.7554/eLife.04490.011",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 11),
            ("ordinal", 6),
            ("sibling_ordinal", 3),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.012"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.012"),
            ("id", "fig4"),
            (
                "title",
                "Manduca sexta larvae grow larger on LOX2/3-deficient plants, regardless of TPS10 expression, in a glasshouse experiment.",
            ),
            (
                "full_title",
                "<italic>Manduca sexta</italic> larvae grow larger on <italic>LOX2/3</italic>-deficient plants, regardless of <italic>TPS10</italic> expression, in a glasshouse experiment.",
            ),
            ("label", "Figure 4."),
            ("full_label", "Figure 4."),
            (
                "caption",
                "Data are shown as mean ± SEM; n = 15–19 larvae on day 8 and 11–16 larvae on day 12. One M. sexta neonate per plant (starting n = 25 larvae) was placed immediately after hatching on the youngest rosette leaf of an elongated plant. Larvae were weighed at the third and fourth instars (of 5 total), corresponding to days 8 and 12, after which larvae become mobile between plants. a,b/A,B Different letters indicate significant differences (corrected p < 0.001) in larval mass on different plant genotypes within each day, in Wilcoxon rank sum tests following significant (corrected p < 0.001) Kruskal–Wallis tests for each day (WT vs lox2/3 day 8, W17,19 = 310, corrected p < 0.001, day 12, W15,15 = 217, corrected p < 0.001; TPS10 vs lox2/3xTPS10, day 8, W15,20 = 290, corrected p < 0.001, day 12, W11,14 = 143, corrected p < 0.001; WT vs TPS10 day 8, W17,15 = 95, corrected p = 0.227, day 12, W15,11 = 56, corrected p = 0.361; lox2/3 vs lox2/3xTPS10 day 8, W19,20 = 249, corrected p = 0.201, day 12, W16,14 = 134, corrected p = 0.377). p-values were corrected for multiple testing using the Holm-Bonferroni method.",
            ),
            (
                "full_caption",
                "Data are shown as mean ± SEM; n = 15–19 larvae on day 8 and 11–16 larvae on day 12. One <italic>M. sexta</italic> neonate per plant (starting n = 25 larvae) was placed immediately after hatching on the youngest rosette leaf of an elongated plant. Larvae were weighed at the third and fourth instars (of 5 total), corresponding to days 8 and 12, after which larvae become mobile between plants. <sup>a,b/A,B</sup> Different letters indicate significant differences (corrected p < 0.001) in larval mass on different plant genotypes within each day, in Wilcoxon rank sum tests following significant (corrected p < 0.001) Kruskal–Wallis tests for each day (WT vs <italic>lox2/3</italic> day 8, W<sub>17,19</sub> = 310, corrected p <italic>&lt;</italic> 0.001, day 12, W<sub>15,15</sub> = 217, corrected p <italic>&lt;</italic> 0.001; <italic>TPS10</italic> vs <italic>lox2/3</italic>x<italic>TPS10</italic>, day 8, W<sub>15,20</sub> = 290, corrected p <italic>&lt;</italic> 0.001, day 12, W<sub>11,14</sub> = 143, corrected p <italic>&lt;</italic> 0.001; WT vs <italic>TPS10</italic> day 8, W<sub>17,15</sub> = 95, corrected p = 0.227, day 12, W<sub>15,11</sub> = 56, corrected p <italic>=</italic> 0.361; <italic>lox2/3</italic> vs <italic>lox2/3</italic>x<italic>TPS10</italic> day 8, W<sub>19,20</sub> = 249, corrected p = 0.201, day 12, W<sub>16,14</sub> = 134, corrected p = 0.377). p<italic>-</italic>values were corrected for multiple testing using the Holm-Bonferroni method.",
            ),
            (
                "content",
                "Data are shown as mean ± SEM; n = 15–19 larvae on day 8 and 11–16 larvae on day 12. One M. sexta neonate per plant (starting n = 25 larvae) was placed immediately after hatching on the youngest rosette leaf of an elongated plant. Larvae were weighed at the third and fourth instars (of 5 total), corresponding to days 8 and 12, after which larvae become mobile between plants. a,b/A,B Different letters indicate significant differences (corrected p < 0.001) in larval mass on different plant genotypes within each day, in Wilcoxon rank sum tests following significant (corrected p < 0.001) Kruskal–Wallis tests for each day (WT vs lox2/3 day 8, W17,19 = 310, corrected p < 0.001, day 12, W15,15 = 217, corrected p < 0.001; TPS10 vs lox2/3xTPS10, day 8, W15,20 = 290, corrected p < 0.001, day 12, W11,14 = 143, corrected p < 0.001; WT vs TPS10 day 8, W17,15 = 95, corrected p = 0.227, day 12, W15,11 = 56, corrected p = 0.361; lox2/3 vs lox2/3xTPS10 day 8, W19,20 = 249, corrected p = 0.201, day 12, W16,14 = 134, corrected p = 0.377). p-values were corrected for multiple testing using the Holm-Bonferroni method. DOI: http://dx.doi.org/10.7554/eLife.04490.012",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 12),
            ("ordinal", 7),
            ("sibling_ordinal", 4),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.013"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.013"),
            ("id", "fig5"),
            (
                "title",
                "Only LOX2/3 and not TPS10, plant position, or population type determined total foliar damage from herbivores over two consecutive field seasons.",
            ),
            (
                "full_title",
                "Only <italic>LOX2/3</italic> and not <italic>TPS10</italic>, plant position, or population type determined total foliar damage from herbivores over two consecutive field seasons.",
            ),
            ("label", "Figure 5."),
            ("full_label", "Figure 5."),
            (
                "caption",
                "Note differences in y-axis scale. Data were collected on June 9th in season one (left panels), and on June 2nd in season two (right panels) from plants at all different positions in the experiment: individual plants (A–B), and plants at the edges (C–D) or at the centers (E–F) of populations. Black bars denote WT, grey bars TPS10, red bars lox2/3, and pink bars lox2/3xTPS10 plants. Total canopy damage reflected the trends in canopy damage caused by individual herbivores (Figure 5—source data 1, 2). Total canopy damage in season one was low (left panels), ranging from 5–10% on lox2/3 and lox2/3xTPS10 plants, and only 2–5% on WT and TPS10 plants. Damage levels were about twice as high in season two (right panels), but showed the same relative pattern, with LOX2/3-deficient plans having more damage. Damage levels within a season were similar for plants in different positions (p > 0.07, see Appendix 3). a,b Different letters indicate significant differences (p < 0.05) between LOX2/3-expressing and LOX2/3-deficient plants (WT, TPS10 v. lox2/3, lox2/3xTPS10) in minimal ANOVA or linear mixed-effects models on arcsin-transformed data. For individuals and center plants n = 7–13, and for edge plants n = 16–48 (up to 4 per population; the blocking effect was accounted for by a random factor in statistical analysis); exact replicate numbers are given in Figure 5—source data 1, 2. There were no significant differences in damage between plants differing only in TPS10 expression (p > 0.4), plants in mono- vs mixed cultures (p > 0.5), or plants of the same genotype at different positions (individual, edge, or center, p > 0.07). Statistical models are given in Appendix 3.",
            ),
            (
                "full_caption",
                'Note differences in y-axis scale. Data were collected on June 9th in season one (left panels), and on June 2nd in season two (right panels) from plants at all different positions in the experiment: individual plants (<bold>A</bold>–<bold>B</bold>), and plants at the edges (<bold>C</bold>–<bold>D</bold>) or at the centers (<bold>E</bold>–<bold>F</bold>) of populations. Black bars denote WT, grey bars <italic>TPS10</italic>, red bars <italic>lox2/3</italic>, and pink bars <italic>lox2/3xTPS10</italic> plants. Total canopy damage reflected the trends in canopy damage caused by individual herbivores (<xref ref-type="supplementary-material" rid="SD4-data SD5-data">Figure 5—source data 1, 2</xref>). Total canopy damage in season one was low (left panels), ranging from 5–10% on <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> plants, and only 2–5% on WT and <italic>TPS10</italic> plants. Damage levels were about twice as high in season two (right panels), but showed the same relative pattern, with <italic>LOX2/3</italic>-deficient plans having more damage. Damage levels within a season were similar for plants in different positions (p > 0.07, see <xref ref-type="app" rid="app3">Appendix 3</xref>). <sup>a,b</sup> Different letters indicate significant differences (p < 0.05) between <italic>LOX2/3</italic>-expressing and <italic>LOX2/3</italic>-deficient plants (WT, <italic>TPS10</italic> v. <italic>lox2/3</italic>, <italic>lox2/3xTPS10</italic>) in minimal ANOVA or linear mixed-effects models on arcsin-transformed data. For individuals and center plants n = 7–13, and for edge plants n = 16–48 (up to 4 per population; the blocking effect was accounted for by a random factor in statistical analysis); exact replicate numbers are given in <xref ref-type="supplementary-material" rid="SD4-data SD5-data">Figure 5—source data 1, 2</xref>. There were no significant differences in damage between plants differing only in <italic>TPS10</italic> expression (p > 0.4), plants in mono- vs mixed cultures (p > 0.5), or plants of the same genotype at different positions (individual, edge, or center, p > 0.07). Statistical models are given in <xref ref-type="app" rid="app3">Appendix 3</xref>.',
            ),
            (
                "content",
                "Note differences in y-axis scale. Data were collected on June 9th in season one (left panels), and on June 2nd in season two (right panels) from plants at all different positions in the experiment: individual plants (A–B), and plants at the edges (C–D) or at the centers (E–F) of populations. Black bars denote WT, grey bars TPS10, red bars lox2/3, and pink bars lox2/3xTPS10 plants. Total canopy damage reflected the trends in canopy damage caused by individual herbivores (Figure 5—source data 1, 2). Total canopy damage in season one was low (left panels), ranging from 5–10% on lox2/3 and lox2/3xTPS10 plants, and only 2–5% on WT and TPS10 plants. Damage levels were about twice as high in season two (right panels), but showed the same relative pattern, with LOX2/3-deficient plans having more damage. Damage levels within a season were similar for plants in different positions (p > 0.07, see Appendix 3). a,b Different letters indicate significant differences (p < 0.05) between LOX2/3-expressing and LOX2/3-deficient plants (WT, TPS10 v. lox2/3, lox2/3xTPS10) in minimal ANOVA or linear mixed-effects models on arcsin-transformed data. For individuals and center plants n = 7–13, and for edge plants n = 16–48 (up to 4 per population; the blocking effect was accounted for by a random factor in statistical analysis); exact replicate numbers are given in Figure 5—source data 1, 2. There were no significant differences in damage between plants differing only in TPS10 expression (p > 0.4), plants in mono- vs mixed cultures (p > 0.5), or plants of the same genotype at different positions (individual, edge, or center, p > 0.07). Statistical models are given in Appendix 3. DOI: http://dx.doi.org/10.7554/eLife.04490.013 10.7554/eLife.04490.014Figure 5—source data 1.Total canopy damage from individual herbivores or groups of herbivores in experimental season one, corresponding to data shown in Figure 5A (mean ± SEM).DOI: http://dx.doi.org/10.7554/eLife.04490.014 DOI: http://dx.doi.org/10.7554/eLife.04490.014 10.7554/eLife.04490.015Figure 5—source data 2.Total canopy damage from individual herbivores or groups of herbivores in experimental season two, corresponding to data shown in Figure 5B (mean ± SEM).DOI: http://dx.doi.org/10.7554/eLife.04490.015 DOI: http://dx.doi.org/10.7554/eLife.04490.015",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 13),
            ("ordinal", 8),
            ("sibling_ordinal", 5),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.014"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.014"),
            ("id", "SD4-data"),
            (
                "title",
                "Total canopy damage from individual herbivores or groups of herbivores in experimental season one, corresponding to data shown in Figure 5A (mean ± SEM).",
            ),
            (
                "full_title",
                'Total canopy damage from individual herbivores or groups of herbivores in experimental season one, corresponding to data shown in <xref ref-type="fig" rid="fig5">Figure 5A</xref> (mean ± SEM).',
            ),
            ("label", "Figure 5—source data 1."),
            ("full_label", "Figure 5—source data 1."),
            ("parent_type", "fig"),
            ("parent_ordinal", 8),
            ("parent_sibling_ordinal", 5),
            ("parent_asset", None),
            ("content", "DOI: http://dx.doi.org/10.7554/eLife.04490.014"),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 14),
            ("ordinal", 4),
            ("sibling_ordinal", 1),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.015"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.015"),
            ("id", "SD5-data"),
            (
                "title",
                "Total canopy damage from individual herbivores or groups of herbivores in experimental season two, corresponding to data shown in Figure 5B (mean ± SEM).",
            ),
            (
                "full_title",
                'Total canopy damage from individual herbivores or groups of herbivores in experimental season two, corresponding to data shown in <xref ref-type="fig" rid="fig5">Figure 5B</xref> (mean ± SEM).',
            ),
            ("label", "Figure 5—source data 2."),
            ("full_label", "Figure 5—source data 2."),
            ("parent_type", "fig"),
            ("parent_ordinal", 8),
            ("parent_sibling_ordinal", 5),
            ("parent_asset", None),
            ("content", "DOI: http://dx.doi.org/10.7554/eLife.04490.015"),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 15),
            ("ordinal", 5),
            ("sibling_ordinal", 2),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.016"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.016"),
            ("id", "fig6"),
            (
                "title",
                "Foliar herbivore abundance is determined by TPS10 expression, lox2/3 expression, and plant position.",
            ),
            (
                "full_title",
                "Foliar herbivore abundance is determined by <italic>TPS10</italic> expression, <italic>lox2/3</italic> expression, and plant position.",
            ),
            ("label", "Figure 6."),
            ("full_label", "Figure 6."),
            (
                "caption",
                "Data were collected on June 26th in season two, when herbivores and predators were more abundant (see Figure 5). Black denotes WT, grey TPS10, red lox2/3, and pink lox2/3xTPS10 plants. Counts for individual herbivores and predators are given in Figure 6—source data 1, 2. (A) Herbivore counts on focal plants of each genotype revealed fewer herbivores present on plants expressing TPS10 (n given in Figure 6—source data 1). a,b Different letters indicate significant differences (corrected p < 0.05) between genotypes in Tukey contrasts following significant differences in a generalized linear mixed-effects model (see Appendix 3). (B) For WT and TPS10 individuals or plants in monocultures (n given in Figure 6—source data 2), there was a significant effect (p < 0.05) of plant genotype (z = −3.662, p = 0.0003) and an interaction of genotype with position (TPS10 by center vs individual position, z = 2.186, p = 0.0288, generalized linear mixed-effects model in Appendix 3). (C) The presence of a TPS10-expressing neighbor in mixed populations did not significantly affect herbivore abundance on WT or lox2/3 edge plants, but there was a significant difference between WT and lox2/3 genotypes at the edges of populations (n given in Figure 6—source data 2, z = 2.358, p = 0.0183, generalized linear model in Appendix 3).",
            ),
            (
                "full_caption",
                'Data were collected on June 26th in season two, when herbivores and predators were more abundant (see <xref ref-type="fig" rid="fig5">Figure 5</xref>). Black denotes WT, grey <italic>TPS10</italic>, red <italic>lox2/3</italic>, and pink <italic>lox2/3xTPS10</italic> plants. Counts for individual herbivores and predators are given in <xref ref-type="supplementary-material" rid="SD6-data SD7-data">Figure 6—source data 1, 2</xref>. (<bold>A</bold>) Herbivore counts on focal plants of each genotype revealed fewer herbivores present on plants expressing <italic>TPS10</italic> (n given in <xref ref-type="supplementary-material" rid="SD6-data">Figure 6—source data 1</xref>). <sup>a,b</sup> Different letters indicate significant differences (corrected p < 0.05) between genotypes in Tukey contrasts following significant differences in a generalized linear mixed-effects model (see <xref ref-type="app" rid="app3">Appendix 3</xref>). (<bold>B</bold>) For WT and <italic>TPS10</italic> individuals or plants in monocultures (n given in <xref ref-type="supplementary-material" rid="SD7-data">Figure 6—source data 2</xref>), there was a significant effect (p < 0.05) of plant genotype (z = −3.662, p = 0.0003) and an interaction of genotype with position (<italic>TPS10</italic> by center vs individual position, z = 2.186, p = 0.0288, generalized linear mixed-effects model in <xref ref-type="app" rid="app3">Appendix 3</xref>). (<bold>C</bold>) The presence of a <italic>TPS10</italic>-expressing neighbor in mixed populations did not significantly affect herbivore abundance on WT or <italic>lox2/3</italic> edge plants, but there was a significant difference between WT and <italic>lox2/3</italic> genotypes at the edges of populations (n given in <xref ref-type="supplementary-material" rid="SD7-data">Figure 6—source data 2</xref>, z = 2.358, p = 0.0183, generalized linear model in <xref ref-type="app" rid="app3">Appendix 3</xref>).',
            ),
            (
                "content",
                "Data were collected on June 26th in season two, when herbivores and predators were more abundant (see Figure 5). Black denotes WT, grey TPS10, red lox2/3, and pink lox2/3xTPS10 plants. Counts for individual herbivores and predators are given in Figure 6—source data 1, 2. (A) Herbivore counts on focal plants of each genotype revealed fewer herbivores present on plants expressing TPS10 (n given in Figure 6—source data 1). a,b Different letters indicate significant differences (corrected p < 0.05) between genotypes in Tukey contrasts following significant differences in a generalized linear mixed-effects model (see Appendix 3). (B) For WT and TPS10 individuals or plants in monocultures (n given in Figure 6—source data 2), there was a significant effect (p < 0.05) of plant genotype (z = −3.662, p = 0.0003) and an interaction of genotype with position (TPS10 by center vs individual position, z = 2.186, p = 0.0288, generalized linear mixed-effects model in Appendix 3). (C) The presence of a TPS10-expressing neighbor in mixed populations did not significantly affect herbivore abundance on WT or lox2/3 edge plants, but there was a significant difference between WT and lox2/3 genotypes at the edges of populations (n given in Figure 6—source data 2, z = 2.358, p = 0.0183, generalized linear model in Appendix 3). DOI: http://dx.doi.org/10.7554/eLife.04490.016 10.7554/eLife.04490.017Figure 6—source data 1.Total and median numbers of herbivores and predators counted on focal plants of different genotypes on June 26th in season two.DOI: http://dx.doi.org/10.7554/eLife.04490.017 DOI: http://dx.doi.org/10.7554/eLife.04490.017 10.7554/eLife.04490.018Figure 6—source data 2.Total and median numbers of herbivores counted on focal plants of different genotypes on June 26th in season two, broken down into different population types and plant locations for WT and TPS10 plants.Replicate numbers of lox2/3 and lox2/3xTPS10 plants were not sufficient for this analysis (n ≤ 3 per population type and position).DOI: http://dx.doi.org/10.7554/eLife.04490.018 Replicate numbers of lox2/3 and lox2/3xTPS10 plants were not sufficient for this analysis (n ≤ 3 per population type and position). DOI: http://dx.doi.org/10.7554/eLife.04490.018",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 16),
            ("ordinal", 9),
            ("sibling_ordinal", 6),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.017"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.017"),
            ("id", "SD6-data"),
            (
                "title",
                "Total and median numbers of herbivores and predators counted on focal plants of different genotypes on June 26th in season two.",
            ),
            (
                "full_title",
                "Total and median numbers of herbivores and predators counted on focal plants of different genotypes on June 26th in season two.",
            ),
            ("label", "Figure 6—source data 1."),
            ("full_label", "Figure 6—source data 1."),
            ("parent_type", "fig"),
            ("parent_ordinal", 9),
            ("parent_sibling_ordinal", 6),
            ("parent_asset", None),
            ("content", "DOI: http://dx.doi.org/10.7554/eLife.04490.017"),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 17),
            ("ordinal", 6),
            ("sibling_ordinal", 1),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.018"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.018"),
            ("id", "SD7-data"),
            (
                "title",
                "Total and median numbers of herbivores counted on focal plants of different genotypes on June 26th in season two, broken down into different population types and plant locations for WT and TPS10 plants.",
            ),
            (
                "full_title",
                "Total and median numbers of herbivores counted on focal plants of different genotypes on June 26th in season two, broken down into different population types and plant locations for WT and <italic>TPS10</italic> plants.",
            ),
            ("label", "Figure 6—source data 2."),
            ("full_label", "Figure 6—source data 2."),
            (
                "caption",
                "Replicate numbers of lox2/3 and lox2/3xTPS10 plants were not sufficient for this analysis (n ≤ 3 per population type and position).",
            ),
            (
                "full_caption",
                "Replicate numbers of <italic>lox2/3</italic> and <italic>lox2/3</italic>x<italic>TPS10</italic> plants were not sufficient for this analysis (n ≤ 3 per population type and position).",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 9),
            ("parent_sibling_ordinal", 6),
            ("parent_asset", None),
            (
                "content",
                "Replicate numbers of lox2/3 and lox2/3xTPS10 plants were not sufficient for this analysis (n ≤ 3 per population type and position). DOI: http://dx.doi.org/10.7554/eLife.04490.018",
            ),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 18),
            ("ordinal", 7),
            ("sibling_ordinal", 2),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.019"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.019"),
            ("id", "fig7"),
            ("title", "LOX2/3 deficiency accelerated flowering under low herbivory."),
            (
                "full_title",
                "<italic>LOX2/3</italic> deficiency accelerated flowering under low herbivory.",
            ),
            ("label", "Figure 7."),
            ("full_label", "Figure 7."),
            (
                "caption",
                "Flower production was monitored twice per season: once shortly after plants began to flower and again 1 week later, when most or all plants were flowering. Bars indicate either total percentage of plants of each genotype which were flowering (left panels), or the mean number of flowers per plant (right panels). WT and TPS10 (black and grey bars) are shown opposite lox2/3 and lox2/3xTPS10 (red and pink bars). Ratios in bars indicate numbers of flowering/total plants monitored; a few plants were not included in counts which had recently lost their main stem to herbivores and had not yet re-grown. (A) In season one, when plants suffered less damage from herbivores (Figure 5), plants with the lox2/3 silencing construct produced more flowers earlier: asterisks indicate pairwise differences between genotypes differing only in the lox2/3 silencing construct. ***Corrected p < 0.001, **corrected p < 0.01, *corrected p < 0.05, or no significant difference (ns) in G-tests (percentage flowering) or in Wilcoxon rank sum tests (flower number, WT vs lox2/3: May 11th, W120,120 = 8582, p = 0.0002; May 17th, W120,119 = 5043, p < 0.0001; TPS10 vs lox2/3xTPS10: May 11th, W84,84 = 3801, p = 0.2063; May 17th, W84,84 = 2520, p = 0.0027). (B) In season two, during which plants received more damage from herbivores (Figure 5), flowering was monitored at later dates: the first timepoint in (B) is comparable to the second timepoint in (A) in terms of the proportion of plants flowering and the average number of flowers per plant (note difference in scale). In season two, LOX2/3 deficiency rather decreased early flower numbers in the comparison of WT vs lox2/3 on May 29th, and did not increase flower numbers at either measurement (WT vs lox2/3: May 29th, W103,95 = 3860.5, p = 0.0091; June 6th, W97,69 = 2940.5, p = 0.3018; TPS10 vs lox2/3xTPS10: May 29th, W87,74 = 3646.5, p = 0.1417; June 6th, W83,63 = 2695, p = 0.7518). For a complete analysis of the effects of LOX2/3 deficiency and TPS10 expression, plant position, and population type on flowering and plant size, see source data file for Figure 8.",
            ),
            (
                "full_caption",
                'Flower production was monitored twice per season: once shortly after plants began to flower and again 1 week later, when most or all plants were flowering. Bars indicate either total percentage of plants of each genotype which were flowering (left panels), or the mean number of flowers per plant (right panels). WT and <italic>TPS10</italic> (black and grey bars) are shown opposite <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> (red and pink bars). Ratios in bars indicate numbers of flowering/total plants monitored; a few plants were not included in counts which had recently lost their main stem to herbivores and had not yet re-grown. (<bold>A</bold>) In season one, when plants suffered less damage from herbivores (<xref ref-type="fig" rid="fig5">Figure 5</xref>), plants with the <italic>lox2/3</italic> silencing construct produced more flowers earlier: asterisks indicate pairwise differences between genotypes differing only in the <italic>lox2/3</italic> silencing construct. ***Corrected p < 0.001, **corrected p < 0.01, *corrected p < 0.05, or no significant difference (ns) in G-tests (percentage flowering) or in Wilcoxon rank sum tests (flower number, WT vs <italic>lox2/3</italic>: May 11th, W<sub>120,120</sub> = 8582, p = 0.0002; May 17th, W<sub>120,119</sub> = 5043, p < 0.0001; <italic>TPS10</italic> vs <italic>lox2/3</italic>x<italic>TPS10</italic>: May 11th, W<sub>84,84</sub> = 3801, p = 0.2063; May 17th, W<sub>84,84</sub> = 2520, p = 0.0027). (<bold>B</bold>) In season two, during which plants received more damage from herbivores (<xref ref-type="fig" rid="fig5">Figure 5</xref>), flowering was monitored at later dates: the first timepoint in (<bold>B</bold>) is comparable to the second timepoint in (<bold>A</bold>) in terms of the proportion of plants flowering and the average number of flowers per plant (note difference in scale). In season two, <italic>LOX2/3</italic> deficiency rather decreased early flower numbers in the comparison of WT vs <italic>lox2/3</italic> on May 29th, and did not increase flower numbers at either measurement (WT vs <italic>lox2/3</italic>: May 29th, W<sub>103,95</sub> = 3860.5, p = 0.0091; June 6th, W<sub>97,69</sub> = 2940.5, p = 0.3018; <italic>TPS10</italic> vs <italic>lox2/3</italic>x<italic>TPS10</italic>: May 29th, W<sub>87,74</sub> = 3646.5, p = 0.1417; June 6th, W<sub>83,63</sub> = 2695, p = 0.7518). For a complete analysis of the effects of <italic>LOX2/3</italic> deficiency and <italic>TPS10</italic> expression, plant position, and population type on flowering and plant size, see source data file for <xref ref-type="fig" rid="fig8">Figure 8</xref>.',
            ),
            (
                "content",
                "Flower production was monitored twice per season: once shortly after plants began to flower and again 1 week later, when most or all plants were flowering. Bars indicate either total percentage of plants of each genotype which were flowering (left panels), or the mean number of flowers per plant (right panels). WT and TPS10 (black and grey bars) are shown opposite lox2/3 and lox2/3xTPS10 (red and pink bars). Ratios in bars indicate numbers of flowering/total plants monitored; a few plants were not included in counts which had recently lost their main stem to herbivores and had not yet re-grown. (A) In season one, when plants suffered less damage from herbivores (Figure 5), plants with the lox2/3 silencing construct produced more flowers earlier: asterisks indicate pairwise differences between genotypes differing only in the lox2/3 silencing construct. ***Corrected p < 0.001, **corrected p < 0.01, *corrected p < 0.05, or no significant difference (ns) in G-tests (percentage flowering) or in Wilcoxon rank sum tests (flower number, WT vs lox2/3: May 11th, W120,120 = 8582, p = 0.0002; May 17th, W120,119 = 5043, p < 0.0001; TPS10 vs lox2/3xTPS10: May 11th, W84,84 = 3801, p = 0.2063; May 17th, W84,84 = 2520, p = 0.0027). (B) In season two, during which plants received more damage from herbivores (Figure 5), flowering was monitored at later dates: the first timepoint in (B) is comparable to the second timepoint in (A) in terms of the proportion of plants flowering and the average number of flowers per plant (note difference in scale). In season two, LOX2/3 deficiency rather decreased early flower numbers in the comparison of WT vs lox2/3 on May 29th, and did not increase flower numbers at either measurement (WT vs lox2/3: May 29th, W103,95 = 3860.5, p = 0.0091; June 6th, W97,69 = 2940.5, p = 0.3018; TPS10 vs lox2/3xTPS10: May 29th, W87,74 = 3646.5, p = 0.1417; June 6th, W83,63 = 2695, p = 0.7518). For a complete analysis of the effects of LOX2/3 deficiency and TPS10 expression, plant position, and population type on flowering and plant size, see source data file for Figure 8. DOI: http://dx.doi.org/10.7554/eLife.04490.019",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 19),
            ("ordinal", 10),
            ("sibling_ordinal", 7),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.020"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.020"),
            ("id", "fig8"),
            (
                "title",
                "TPS10 expression reduced flower production under high herbivory.",
            ),
            (
                "full_title",
                "<italic>TPS10</italic> expression reduced flower production under high herbivory.",
            ),
            ("label", "Figure 8."),
            ("full_label", "Figure 8."),
            (
                "caption",
                "Plant size as measured by stem length (left panels), and reproduction as measured by flower production (right panels); n given in Figure 8—source data 1. Data were collected on June 6th in season two from plants at all different positions in the experiment: individual plants (A–B), and plants at the edges (C–D) or at the centers (E–F) of populations. Black bars denote WT, grey bars TPS10, red bars lox2/3, and pink bars lox2/3xTPS10 plants. Plant height differed little, with only a small negative effect of TPS10 expression for center plants (E), which was more pronounced in mixed- than monocultures, indicating a slight competitive disadvantage for these plants. Individual plants differed much more than plants in populations in terms of flower production, with TPS10-expressing plants at a disadvantage (B); this effect was greatly reduced in populations.a,b Different letters indicate significant differences (corrected p < 0.05) in Tukey post-hoc tests on plant genotype following significant effects in a generalized linear model, or generalized linear mixed-effects model (C–D) with genotype as a factor; these p-values were corrected using the Holm-Bonferroni correction for multiple testing as the same data were also tested for effects of LOX2/3 deficiency and TPS10 expression (see Figure 8—source data 1). There was no difference in flower production for edge plants, but a small negative effect of TPS10 expression on flower production in center plants (F) corresponding to the slight reduction in height of these plants (E). Statistical models are given in Figure 8—source data 1.",
            ),
            (
                "full_caption",
                'Plant size as measured by stem length (left panels), and reproduction as measured by flower production (right panels); n given in <xref ref-type="supplementary-material" rid="SD8-data">Figure 8—source data 1</xref>. Data were collected on June 6th in season two from plants at all different positions in the experiment: individual plants (<bold>A</bold>–<bold>B</bold>), and plants at the edges (<bold>C</bold>–<bold>D</bold>) or at the centers (<bold>E</bold>–<bold>F</bold>) of populations. Black bars denote WT, grey bars <italic>TPS10</italic>, red bars <italic>lox2/3</italic>, and pink bars <italic>lox2/3xTPS10</italic> plants. Plant height differed little, with only a small negative effect of <italic>TPS10</italic> expression for center plants (<bold>E</bold>), which was more pronounced in mixed- than monocultures, indicating a slight competitive disadvantage for these plants. Individual plants differed much more than plants in populations in terms of flower production, with <italic>TPS10</italic>-expressing plants at a disadvantage (<bold>B</bold>); this effect was greatly reduced in populations.<sup>a,b</sup> Different letters indicate significant differences (corrected p < 0.05) in Tukey post-hoc tests on plant genotype following significant effects in a generalized linear model, or generalized linear mixed-effects model (<bold>C</bold>–<bold>D</bold>) with genotype as a factor; these p-values were corrected using the Holm-Bonferroni correction for multiple testing as the same data were also tested for effects of <italic>LOX2/3</italic> deficiency and <italic>TPS10</italic> expression (see <xref ref-type="supplementary-material" rid="SD8-data">Figure 8—source data 1</xref>). There was no difference in flower production for edge plants, but a small negative effect of <italic>TPS10</italic> expression on flower production in center plants (<bold>F</bold>) corresponding to the slight reduction in height of these plants (<bold>E</bold>). Statistical models are given in <xref ref-type="supplementary-material" rid="SD8-data">Figure 8—source data 1</xref>.',
            ),
            (
                "content",
                "Plant size as measured by stem length (left panels), and reproduction as measured by flower production (right panels); n given in Figure 8—source data 1. Data were collected on June 6th in season two from plants at all different positions in the experiment: individual plants (A–B), and plants at the edges (C–D) or at the centers (E–F) of populations. Black bars denote WT, grey bars TPS10, red bars lox2/3, and pink bars lox2/3xTPS10 plants. Plant height differed little, with only a small negative effect of TPS10 expression for center plants (E), which was more pronounced in mixed- than monocultures, indicating a slight competitive disadvantage for these plants. Individual plants differed much more than plants in populations in terms of flower production, with TPS10-expressing plants at a disadvantage (B); this effect was greatly reduced in populations.a,b Different letters indicate significant differences (corrected p < 0.05) in Tukey post-hoc tests on plant genotype following significant effects in a generalized linear model, or generalized linear mixed-effects model (C–D) with genotype as a factor; these p-values were corrected using the Holm-Bonferroni correction for multiple testing as the same data were also tested for effects of LOX2/3 deficiency and TPS10 expression (see Figure 8—source data 1). There was no difference in flower production for edge plants, but a small negative effect of TPS10 expression on flower production in center plants (F) corresponding to the slight reduction in height of these plants (E). Statistical models are given in Figure 8—source data 1. DOI: http://dx.doi.org/10.7554/eLife.04490.020 10.7554/eLife.04490.021Figure 8—source data 1.The same statistical approach described in Appendix 3 for Figure 5 was used for plant size and reproduction data from May 17th in season 1, and June 6th in season two (the most complete measurements, closest to the date of herbivore damage screens, from each season).It should be kept in mind that plants in the season one analysis were at an earlier stage of growth and flowering, for example, had about 10% as many flowers as plants in the season 2 analysis; for an analysis of flowering at similar stages, see Figure 7. Significant p-values and factors are given in bold, while p-values and factors marked in grey and in bold are marginal, but are required in the minimal model because their removal resulted in poorer (higher) AIC values. Other non-significant factors remained in models because they were the least insignificant factor (if only one factor remained), or because they are dependent on other factors which are significant or required.DOI: http://dx.doi.org/10.7554/eLife.04490.021 It should be kept in mind that plants in the season one analysis were at an earlier stage of growth and flowering, for example, had about 10% as many flowers as plants in the season 2 analysis; for an analysis of flowering at similar stages, see Figure 7. Significant p-values and factors are given in bold, while p-values and factors marked in grey and in bold are marginal, but are required in the minimal model because their removal resulted in poorer (higher) AIC values. Other non-significant factors remained in models because they were the least insignificant factor (if only one factor remained), or because they are dependent on other factors which are significant or required. DOI: http://dx.doi.org/10.7554/eLife.04490.021",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 20),
            ("ordinal", 11),
            ("sibling_ordinal", 8),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.021"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.021"),
            ("id", "SD8-data"),
            (
                "title",
                "The same statistical approach described in Appendix 3 for Figure 5 was used for plant size and reproduction data from May 17th in season 1, and June 6th in season two (the most complete measurements, closest to the date of herbivore damage screens, from each season).",
            ),
            (
                "full_title",
                'The same statistical approach described in <xref ref-type="app" rid="app3">Appendix 3</xref> for <xref ref-type="fig" rid="fig5">Figure 5</xref> was used for plant size and reproduction data from May 17th in season 1, and June 6th in season two (the most complete measurements, closest to the date of herbivore damage screens, from each season).',
            ),
            ("label", "Figure 8—source data 1."),
            ("full_label", "Figure 8—source data 1."),
            (
                "caption",
                "It should be kept in mind that plants in the season one analysis were at an earlier stage of growth and flowering, for example, had about 10% as many flowers as plants in the season 2 analysis; for an analysis of flowering at similar stages, see Figure 7. Significant p-values and factors are given in bold, while p-values and factors marked in grey and in bold are marginal, but are required in the minimal model because their removal resulted in poorer (higher) AIC values. Other non-significant factors remained in models because they were the least insignificant factor (if only one factor remained), or because they are dependent on other factors which are significant or required.",
            ),
            (
                "full_caption",
                'It should be kept in mind that plants in the season one analysis were at an earlier stage of growth and flowering, for example, had about 10% as many flowers as plants in the season 2 analysis; for an analysis of flowering at similar stages, see <xref ref-type="fig" rid="fig7">Figure 7</xref>. Significant p-values and factors are given in bold, while p-values and factors marked in grey and in bold are marginal, but are required in the minimal model because their removal resulted in poorer (higher) AIC values. Other non-significant factors remained in models because they were the least insignificant factor (if only one factor remained), or because they are dependent on other factors which are significant or required.',
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 11),
            ("parent_sibling_ordinal", 8),
            ("parent_asset", None),
            (
                "content",
                "It should be kept in mind that plants in the season one analysis were at an earlier stage of growth and flowering, for example, had about 10% as many flowers as plants in the season 2 analysis; for an analysis of flowering at similar stages, see Figure 7. Significant p-values and factors are given in bold, while p-values and factors marked in grey and in bold are marginal, but are required in the minimal model because their removal resulted in poorer (higher) AIC values. Other non-significant factors remained in models because they were the least insignificant factor (if only one factor remained), or because they are dependent on other factors which are significant or required. DOI: http://dx.doi.org/10.7554/eLife.04490.021",
            ),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 21),
            ("ordinal", 8),
            ("sibling_ordinal", 1),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.022"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.022"),
            ("id", "fig9"),
            (
                "title",
                "One TPS10-expressing plant per population counteracts a mortality increase in LOX2/3-deficient populations.",
            ),
            (
                "full_title",
                "One <italic>TPS10</italic>-expressing plant per population counteracts a mortality increase in <italic>LOX2/3</italic>-deficient populations.",
            ),
            ("label", "Figure 9."),
            ("full_label", "Figure 9."),
            (
                "caption",
                "Bars indicate total percentage mortality for all plants in each population type, and ratios in bars indicate numbers of dead/total plants (n = 75). WT and TPS10 populations (left, black, and grey bars) are compared to the equivalent lox2/3 and lox2/3xTPS10 populations (right, red, and pink bars) differing only in the lox2/3 silencing construct. LOX2/3-deficient plants had higher mortality in monocultures, regardless of whether they also expressed TPS10 (**WT mono v. lox2/3 mono, corrected p < 0.01; *TPS10 mono v. lox2/3xTPS10 mono, corrected p < 0.05 in G-tests; p-values were corrected for multiple testing using the Holm-Bonferroni method). However, the mortality of plants in mixed lox2/3 + lox2/3xTPS10 populations is similar to that of the plants in mixed WT + TPS10 populations (ns, not significant). There was no significant difference in mortality among mono- and mixed cultures of WT and TPS10 plants (corrected p > 0.3), or of lox2/3 and lox2/3xTPS10 plants, although there was a marginal difference between lox2/3 monocultures and lox2/3 + lox2/3xTPS10 mixed cultures (corrected p = 0.091). In addition, a health index was assigned to each plant (discussed in text); this index was correlated with several plant growth and reproduction parameters (see Figure 9—figure supplement 1). Data were collected on June 29th at the end of experimental season two; in season one, all plants experienced much lower herbivory (Figure 5) and negligible mortality (0–5%).",
            ),
            (
                "full_caption",
                'Bars indicate total percentage mortality for all plants in each population type, and ratios in bars indicate numbers of dead/total plants (n = 75). WT and <italic>TPS10</italic> populations (left, black, and grey bars) are compared to the equivalent <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> populations (right, red, and pink bars) differing only in the <italic>lox2/3</italic> silencing construct. <italic>LOX2/3</italic>-deficient plants had higher mortality in monocultures, regardless of whether they also expressed <italic>TPS10</italic> (**WT mono v. <italic>lox2/3</italic> mono, corrected p < 0.01; *<italic>TPS10</italic> mono v. <italic>lox2/3xTPS10</italic> mono, corrected p < 0.05 in G-tests; p<italic>-</italic>values were corrected for multiple testing using the Holm-Bonferroni method). However, the mortality of plants in mixed <italic>lox2/3</italic> + <italic>lox2/3xTPS10</italic> populations is similar to that of the plants in mixed WT + <italic>TPS10</italic> populations (ns, not significant). There was no significant difference in mortality among mono- and mixed cultures of WT and <italic>TPS10</italic> plants (corrected p > 0.3), or of <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> plants, although there was a marginal difference between <italic>lox2/3</italic> monocultures and <italic>lox2/3</italic> + <italic>lox2/3xTPS10</italic> mixed cultures (corrected p = 0.091). In addition, a health index was assigned to each plant (discussed in text); this index was correlated with several plant growth and reproduction parameters (see <xref ref-type="fig" rid="fig9s1">Figure 9—figure supplement 1</xref>). Data were collected on June 29th at the end of experimental season two; in season one, all plants experienced much lower herbivory (<xref ref-type="fig" rid="fig5">Figure 5</xref>) and negligible mortality (0–5%).',
            ),
            (
                "content",
                "Bars indicate total percentage mortality for all plants in each population type, and ratios in bars indicate numbers of dead/total plants (n = 75). WT and TPS10 populations (left, black, and grey bars) are compared to the equivalent lox2/3 and lox2/3xTPS10 populations (right, red, and pink bars) differing only in the lox2/3 silencing construct. LOX2/3-deficient plants had higher mortality in monocultures, regardless of whether they also expressed TPS10 (**WT mono v. lox2/3 mono, corrected p < 0.01; *TPS10 mono v. lox2/3xTPS10 mono, corrected p < 0.05 in G-tests; p-values were corrected for multiple testing using the Holm-Bonferroni method). However, the mortality of plants in mixed lox2/3 + lox2/3xTPS10 populations is similar to that of the plants in mixed WT + TPS10 populations (ns, not significant). There was no significant difference in mortality among mono- and mixed cultures of WT and TPS10 plants (corrected p > 0.3), or of lox2/3 and lox2/3xTPS10 plants, although there was a marginal difference between lox2/3 monocultures and lox2/3 + lox2/3xTPS10 mixed cultures (corrected p = 0.091). In addition, a health index was assigned to each plant (discussed in text); this index was correlated with several plant growth and reproduction parameters (see Figure 9—figure supplement 1). Data were collected on June 29th at the end of experimental season two; in season one, all plants experienced much lower herbivory (Figure 5) and negligible mortality (0–5%). DOI: http://dx.doi.org/10.7554/eLife.04490.022",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 22),
            ("ordinal", 12),
            ("sibling_ordinal", 9),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.023"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.023"),
            ("id", "fig9s1"),
            (
                "title",
                "The health index assigned to plants is strongly and significantly correlated to several measures of plant size and reproduction.",
            ),
            (
                "full_title",
                "The health index assigned to plants is strongly and significantly correlated to several measures of plant size and reproduction.",
            ),
            ("label", "Figure 9—figure supplement 1."),
            ("full_label", "Figure 9—figure supplement 1."),
            (
                "caption",
                "All measures of plant size and reproduction made on June 6th in season two are plotted against simultaneously assigned health indices >1 (a health index of 1 refers to dead plants). Spearman's ρ and the corresponding Holm-Bonferroni corrected p-value are shown for each correlation.",
            ),
            (
                "full_caption",
                "All measures of plant size and reproduction made on June 6th in season two are plotted against simultaneously assigned health indices >1 (a health index of 1 refers to dead plants). Spearman's ρ and the corresponding Holm-Bonferroni corrected p<italic>-</italic>value are shown for each correlation.",
            ),
            ("parent_type", "fig"),
            ("parent_ordinal", 12),
            ("parent_sibling_ordinal", 9),
            ("parent_asset", None),
            (
                "content",
                "All measures of plant size and reproduction made on June 6th in season two are plotted against simultaneously assigned health indices >1 (a health index of 1 refers to dead plants). Spearman's ρ and the corresponding Holm-Bonferroni corrected p-value are shown for each correlation. DOI: http://dx.doi.org/10.7554/eLife.04490.023",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 23),
            ("ordinal", 13),
            ("sibling_ordinal", 2),
            ("asset", "figsupp"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.024"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.024"),
            ("id", "fig10"),
            (
                "title",
                "TPS10 plants increase the infestation rates of their WT neighbors with the stem-boring weevil T. mucorea.",
            ),
            (
                "full_title",
                "<italic>TPS10</italic> plants increase the infestation rates of their WT neighbors with the stem-boring weevil <italic>T. mucorea.</italic>",
            ),
            ("label", "Figure 10."),
            ("full_label", "Figure 10."),
            (
                "caption",
                "(A) Photograph of a T. mucorea adult emerging from the stem which it infested as a larva reprinted with permission from Anke Steppuhn, Copyright 2005. All rights reserved. Adults mate and oviposit on young N. attenuata plants in March and April, and upon hatching, larvae burrow into the growing stem of the plant chosen by their mother (Diezel et al., 2011). Infestation was scored as the number of plants with hollow, frass-filled stems (from which larvae were usually also recovered). (B) T. mucorea infestation more than doubled for WT plants, but did not change for TPS10 plants in mixed cultures vs monocultures. Bars indicate total percentage of plants infested; ratios in bars indicate numbers of infested/total plants. Data were collected at the end of experimental season two (June 29th). *Corrected p < 0.05 in a Fisher's exact test, n = 31–47 (all surviving replicates). p-values were corrected for multiple testing using the Holm-Bonferroni method. (C) The presence of lox2/3xTPS10 plants also tended to increase T. mucorea infestation in lox2/3 + lox2/3xTPS10 populations but, likely due to lower replicate numbers caused by higher mortality for plants in lox2/3 and lox2/3xTPS10 monocultures (Figure 9), differences are not significant (corrected p-values = 1).",
            ),
            (
                "full_caption",
                '(<bold>A</bold>) Photograph of a <italic>T. mucorea</italic> adult emerging from the stem which it infested as a larva reprinted with permission from Anke Steppuhn, Copyright 2005. All rights reserved. Adults mate and oviposit on young <italic>N. attenuata</italic> plants in March and April, and upon hatching, larvae burrow into the growing stem of the plant chosen by their mother (<xref ref-type="bibr" rid="bib22">Diezel et al., 2011</xref>). Infestation was scored as the number of plants with hollow, frass-filled stems (from which larvae were usually also recovered). (<bold>B</bold>) <italic>T. mucorea</italic> infestation more than doubled for WT plants, but did not change for <italic>TPS10</italic> plants in mixed cultures vs monocultures. Bars indicate total percentage of plants infested; ratios in bars indicate numbers of infested/total plants. Data were collected at the end of experimental season two (June 29th). *Corrected p < 0.05 in a Fisher\'s exact test, n = 31–47 (all surviving replicates). p<italic>-</italic>values were corrected for multiple testing using the Holm-Bonferroni method. (<bold>C</bold>) The presence of <italic>lox2/3xTPS10</italic> plants also tended to increase <italic>T. mucorea</italic> infestation in <italic>lox2/3</italic> + <italic>lox2/3xTPS10</italic> populations but, likely due to lower replicate numbers caused by higher mortality for plants in <italic>lox2/3</italic> and <italic>lox2/3xTPS10</italic> monocultures (<xref ref-type="fig" rid="fig9">Figure 9</xref>), differences are not significant (corrected p<italic>-</italic>values = 1).',
            ),
            (
                "content",
                "(A) Photograph of a T. mucorea adult emerging from the stem which it infested as a larva reprinted with permission from Anke Steppuhn, Copyright 2005. All rights reserved. Adults mate and oviposit on young N. attenuata plants in March and April, and upon hatching, larvae burrow into the growing stem of the plant chosen by their mother (Diezel et al., 2011). Infestation was scored as the number of plants with hollow, frass-filled stems (from which larvae were usually also recovered). (B) T. mucorea infestation more than doubled for WT plants, but did not change for TPS10 plants in mixed cultures vs monocultures. Bars indicate total percentage of plants infested; ratios in bars indicate numbers of infested/total plants. Data were collected at the end of experimental season two (June 29th). *Corrected p < 0.05 in a Fisher's exact test, n = 31–47 (all surviving replicates). p-values were corrected for multiple testing using the Holm-Bonferroni method. (C) The presence of lox2/3xTPS10 plants also tended to increase T. mucorea infestation in lox2/3 + lox2/3xTPS10 populations but, likely due to lower replicate numbers caused by higher mortality for plants in lox2/3 and lox2/3xTPS10 monocultures (Figure 9), differences are not significant (corrected p-values = 1). DOI: http://dx.doi.org/10.7554/eLife.04490.024",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 24),
            ("ordinal", 14),
            ("sibling_ordinal", 10),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.025"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.025"),
            ("id", "fig11"),
            (
                "title",
                "TPS10 and LOX2/3 interact to alter the predation of M. sexta by Geocoris spp from edge vs center plants.",
            ),
            (
                "full_title",
                "<italic>TPS10</italic> and <italic>LOX2/3</italic> interact to alter the predation of <italic>M. sexta</italic> by <italic>Geocoris</italic> spp from edge vs center plants.",
            ),
            ("label", "Figure 11."),
            ("full_label", "Figure 11."),
            (
                "caption",
                "(A) Photograph of a M. sexta egg and first-instar larva, and G. pallens adult on an N. attenuata leaf reprinted with permission from Danny Kessler, Copyright 2006. All rights reserved. As shown here, one first-instar M. sexta larva and one egg were placed on a lower stem leaf in a standardized position on healthy, size-matched plants (one leaf/plant), and their predation was monitored for 24 h. Data are total numbers from five consecutive trials conducted between June 21st and June 27th at the end of experimental season two, when Geocoris spp. and herbivores were more abundant (see Figure 5). (B and C) Predation rates of M. sexta are higher on lox2/3xTPS10 than lox2/3 plants at the edges of monocultures (B), but lower on TPS10 than WT plants at the centers of monocultures (C). Bars indicate the total percentage of M. sexta eggs and larvae predated per population; numbers of predated/total eggs and larvae are given inside bars. The same trends were observed in both egg and larva predation; separate numbers of larvae and eggs predated are given in Figure 11—source data 1. Plants of the WT background and plants with the lox2/3 silencing construct were separately matched for parallel experiments and thus analyzed separately. (B) Predation of herbivores from edge plants was similar (not significant, ns) regardless of the number of TPS10 plants in WT + TPS10 populations, but increased with increasing numbers of lox2/3xTPS10 plants in lox2/3 + lox2/3xTPS10 populations. For plants at the center of populations, the pattern was reversed: (C) predation rates decreased with increasing numbers of TPS10 plants in WT + TPS10 populations, and were similar (ns) regardless of the number of lox2/3xTPS10 plants in lox2/3 + lox2/3xTPS10 populations. a,b Different letters indicate significant differences (corrected p < 0.05) in Fisher's exact tests; p-values were corrected for multiple testing using the Holm-Bonferroni method.",
            ),
            (
                "full_caption",
                '(<bold>A</bold>) Photograph of a <italic>M. sexta</italic> egg and first-instar larva, and <italic>G. pallens</italic> adult on an <italic>N. attenuata</italic> leaf reprinted with permission from Danny Kessler, Copyright 2006. All rights reserved. As shown here, one first-instar <italic>M</italic>. <italic>sexta</italic> larva and one egg were placed on a lower stem leaf in a standardized position on healthy, size-matched plants (one leaf/plant), and their predation was monitored for 24 h. Data are total numbers from five consecutive trials conducted between June 21st and June 27th at the end of experimental season two, when <italic>Geocoris</italic> spp. and herbivores were more abundant (see <xref ref-type="fig" rid="fig5">Figure 5</xref>). (<bold>B</bold> and <bold>C</bold>) Predation rates of <italic>M</italic>. <italic>sexta</italic> are higher on <italic>lox2/3xTPS10</italic> than <italic>lox2/3</italic> plants at the edges of monocultures (<bold>B</bold>), but lower on <italic>TPS10</italic> than WT plants at the centers of monocultures (<bold>C</bold>). Bars indicate the total percentage of <italic>M</italic>. <italic>sexta</italic> eggs and larvae predated per population; numbers of predated/total eggs and larvae are given inside bars. The same trends were observed in both egg and larva predation; separate numbers of larvae and eggs predated are given in <xref ref-type="supplementary-material" rid="SD9-data">Figure 11—source data 1</xref>. Plants of the WT background and plants with the <italic>lox2/3</italic> silencing construct were separately matched for parallel experiments and thus analyzed separately. (<bold>B</bold>) Predation of herbivores from edge plants was similar (not significant, ns) regardless of the number of <italic>TPS10</italic> plants in WT + <italic>TPS10</italic> populations, but increased with increasing numbers of <italic>lox2/3xTPS10</italic> plants in <italic>lox2/3</italic> + <italic>lox2/3xTPS10</italic> populations. For plants at the center of populations, the pattern was reversed: (<bold>C</bold>) predation rates decreased with increasing numbers of <italic>TPS10</italic> plants in WT + <italic>TPS10</italic> populations, and were similar (ns) regardless of the number of <italic>lox2/3xTPS10</italic> plants in <italic>lox2/3</italic> + <italic>lox2/3xTPS10</italic> populations. <sup>a,b</sup> Different letters indicate significant differences (corrected p < 0.05) in Fisher\'s exact tests; p<italic>-</italic>values were corrected for multiple testing using the Holm-Bonferroni method.',
            ),
            (
                "content",
                "(A) Photograph of a M. sexta egg and first-instar larva, and G. pallens adult on an N. attenuata leaf reprinted with permission from Danny Kessler, Copyright 2006. All rights reserved. As shown here, one first-instar M. sexta larva and one egg were placed on a lower stem leaf in a standardized position on healthy, size-matched plants (one leaf/plant), and their predation was monitored for 24 h. Data are total numbers from five consecutive trials conducted between June 21st and June 27th at the end of experimental season two, when Geocoris spp. and herbivores were more abundant (see Figure 5). (B and C) Predation rates of M. sexta are higher on lox2/3xTPS10 than lox2/3 plants at the edges of monocultures (B), but lower on TPS10 than WT plants at the centers of monocultures (C). Bars indicate the total percentage of M. sexta eggs and larvae predated per population; numbers of predated/total eggs and larvae are given inside bars. The same trends were observed in both egg and larva predation; separate numbers of larvae and eggs predated are given in Figure 11—source data 1. Plants of the WT background and plants with the lox2/3 silencing construct were separately matched for parallel experiments and thus analyzed separately. (B) Predation of herbivores from edge plants was similar (not significant, ns) regardless of the number of TPS10 plants in WT + TPS10 populations, but increased with increasing numbers of lox2/3xTPS10 plants in lox2/3 + lox2/3xTPS10 populations. For plants at the center of populations, the pattern was reversed: (C) predation rates decreased with increasing numbers of TPS10 plants in WT + TPS10 populations, and were similar (ns) regardless of the number of lox2/3xTPS10 plants in lox2/3 + lox2/3xTPS10 populations. a,b Different letters indicate significant differences (corrected p < 0.05) in Fisher's exact tests; p-values were corrected for multiple testing using the Holm-Bonferroni method. DOI: http://dx.doi.org/10.7554/eLife.04490.025 10.7554/eLife.04490.026Figure 11—source data 1.Total numbers of M. sexta larvae and eggs predated by Geocoris spp. in experimental season two (corresponding to data shown in Figure 10).DOI: http://dx.doi.org/10.7554/eLife.04490.026 DOI: http://dx.doi.org/10.7554/eLife.04490.026",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 25),
            ("ordinal", 15),
            ("sibling_ordinal", 11),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.026"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.026"),
            ("id", "SD9-data"),
            (
                "title",
                "Total numbers of M. sexta larvae and eggs predated by Geocoris spp. in experimental season two (corresponding to data shown in Figure 10).",
            ),
            (
                "full_title",
                'Total numbers of <italic>M. sexta</italic> larvae and eggs predated by <italic>Geocoris</italic> spp. in experimental season two (corresponding to data shown in <xref ref-type="fig" rid="fig10">Figure 10</xref>).',
            ),
            ("label", "Figure 11—source data 1."),
            ("full_label", "Figure 11—source data 1."),
            ("parent_type", "fig"),
            ("parent_ordinal", 15),
            ("parent_sibling_ordinal", 11),
            ("parent_asset", None),
            ("content", "DOI: http://dx.doi.org/10.7554/eLife.04490.026"),
            ("mimetype", "application"),
            ("mime-subtype", "xlsx"),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "supplementary-material"),
            ("position", 26),
            ("ordinal", 9),
            ("sibling_ordinal", 1),
            ("asset", "data"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.027"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.027"),
            ("id", "fig12"),
            (
                "title",
                "Transgene insertions and reduced jasmonate accumulation in lines of lox2/3.",
            ),
            (
                "full_title",
                "Transgene insertions and reduced jasmonate accumulation in lines of <italic>lox2/3.</italic>",
            ),
            ("label", "Appendix Figure 1."),
            ("full_label", "Appendix Figure 1."),
            (
                "caption",
                "(A) Southern blot of genomic DNA from multiple lines of lox2/3 digested with the restriction enzymes XbaI or EcoRI. Line 707 is highlighted: this line was used for experiments. A single transgene insertion in TPS10 (line 10-3) is shown in Schuman et al. (2014). All lines of lox2/3 screened had strong reductions in their induced (B) JA and (C) JA-Ile accumulation (mean ± SEM, n = 6). Line 707-2 was used for further experiments. a,b Different letters indicate significant differences (corrected p < 0.0001) in Tukey HSD tests following significant (p < 0.0001) 1-way ANOVAs on natural log-transformed data (JA: F4,25 = 54.15; JA-Ile: F4,25 = 18.00); ns, not significant in 1-way ANOVAs (p > 0.4).",
            ),
            (
                "full_caption",
                '(<bold>A</bold>) Southern blot of genomic DNA from multiple lines of <italic>lox2/3</italic> digested with the restriction enzymes XbaI or EcoRI. Line 707 is highlighted: this line was used for experiments. A single transgene insertion in <italic>TPS10</italic> (line 10-3) is shown in <xref ref-type="bibr" rid="bib71">Schuman et al. (2014)</xref>. All lines of <italic>lox2/3</italic> screened had strong reductions in their induced (<bold>B</bold>) JA and (<bold>C</bold>) JA-Ile accumulation (mean ± SEM, n = 6). Line 707-2 was used for further experiments. <sup>a,b</sup> Different letters indicate significant differences (corrected p < 0.0001) in Tukey HSD tests following significant (p < 0.0001) 1-way ANOVAs on natural log-transformed data (JA: F<sub>4,25</sub> = 54.15; JA-Ile: F<sub>4,25</sub> = 18.00); ns, not significant in 1-way ANOVAs (p > 0.4).',
            ),
            (
                "content",
                "(A) Southern blot of genomic DNA from multiple lines of lox2/3 digested with the restriction enzymes XbaI or EcoRI. Line 707 is highlighted: this line was used for experiments. A single transgene insertion in TPS10 (line 10-3) is shown in Schuman et al. (2014). All lines of lox2/3 screened had strong reductions in their induced (B) JA and (C) JA-Ile accumulation (mean ± SEM, n = 6). Line 707-2 was used for further experiments. a,b Different letters indicate significant differences (corrected p < 0.0001) in Tukey HSD tests following significant (p < 0.0001) 1-way ANOVAs on natural log-transformed data (JA: F4,25 = 54.15; JA-Ile: F4,25 = 18.00); ns, not significant in 1-way ANOVAs (p > 0.4). DOI: http://dx.doi.org/10.7554/eLife.04490.027",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 27),
            ("ordinal", 16),
            ("sibling_ordinal", 1),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.028"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.028"),
            ("id", "fig13"),
            (
                "title",
                "Target gene transcript levels in lox2/3, TPS10, and lox2/3xTPS10 plants.",
            ),
            (
                "full_title",
                "Target gene transcript levels in <italic>lox2/3</italic>, <italic>TPS10</italic>, and <italic>lox2/3</italic>x<italic>TPS10</italic> plants.",
            ),
            ("label", "Appendix Figure 2."),
            ("full_label", "Appendix Figure 2."),
            (
                "caption",
                "Transcript accumulation shows that lox2/3 plants, TPS10 plants, and hemizygous crosses (lox2/3xTPS10) have the expected silencing or accumulation of the target genes LOX2, LOX3, and TPS10 (mean ± SEM, n = 4). a,b Different letters indicate significant differences (corrected p < 0.01) in Tukey HSD tests following a significant (p < 0.001) 1-way ANOVA on natural log-transformed data (LOX2: F3,12 = 49.49, p < 0.001; LOX3: F3,12 = 13.73, p < 0.001; TPS10: F3,12 = 845.6, p < 0.001).",
            ),
            (
                "full_caption",
                "Transcript accumulation shows that <italic>lox2/3</italic> plants, <italic>TPS10</italic> plants, and hemizygous crosses (<italic>lox2/3xTPS10</italic>) have the expected silencing or accumulation of the target genes <italic>LOX2</italic>, <italic>LOX3</italic>, and <italic>TPS10</italic> (mean ± SEM, n = 4). <sup>a,b</sup> Different letters indicate significant differences (corrected p < 0.01) in Tukey HSD tests following a significant (p < 0.001) 1-way ANOVA on natural log-transformed data (<italic>LOX2</italic>: F<sub>3,12</sub> = 49.49, p < 0.001; <italic>LOX3</italic>: F<sub>3,12</sub> = 13.73, p < 0.001; <italic>TPS10</italic>: F<sub>3,12</sub> = 845.6, p < 0.001).",
            ),
            (
                "content",
                "Transcript accumulation shows that lox2/3 plants, TPS10 plants, and hemizygous crosses (lox2/3xTPS10) have the expected silencing or accumulation of the target genes LOX2, LOX3, and TPS10 (mean ± SEM, n = 4). a,b Different letters indicate significant differences (corrected p < 0.01) in Tukey HSD tests following a significant (p < 0.001) 1-way ANOVA on natural log-transformed data (LOX2: F3,12 = 49.49, p < 0.001; LOX3: F3,12 = 13.73, p < 0.001; TPS10: F3,12 = 845.6, p < 0.001). DOI: http://dx.doi.org/10.7554/eLife.04490.028",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "fig"),
            ("position", 28),
            ("ordinal", 17),
            ("sibling_ordinal", 2),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.029"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.029"),
            ("id", "tbl1"),
            ("label", "Appendix Table 1."),
            ("full_label", "Appendix Table 1."),
            (
                "caption",
                "Volatiles (percent IS plant−1, mean ± SEM) trapped in the headspace around single plants in experimental season one (June 8th–9th).",
            ),
            (
                "full_caption",
                "Volatiles (percent IS plant<sup>−1</sup>, mean ± SEM) trapped in the headspace around single plants in experimental season one (June 8th–9th).",
            ),
            (
                "content",
                "Volatiles (percent IS plant−1, mean ± SEM) trapped in the headspace around single plants in experimental season one (June 8th–9th). DOI: http://dx.doi.org/10.7554/eLife.04490.029 a, b Different letters indicate significant differences between lines for a vertical category (corrected P<0.05) in Wilcoxon rank sum tests following significant Kruskal-Wallis tests across all genotypes (see Results text); P-values were corrected for multiple testing using the Holm-Bonferroni method. Community type was also tested and found not significant. Where there are no letters, there are no significant pairwise differences within a category. } alox, } blox / } aTPS, } bTPS Indicate significant differences (corrected P<0.05) between lines with and without either the lox construct (WT and TPS v. lox and loxTPS) or the TPS construct (WT and lox v. TPS and loxTPS) in Wilcoxon rank sum tests for a category; P-values were corrected for multiple testing using the Holm-Bonferroni method. There are no significant pairwise differences in these categories. IS, internal standard; GLVs, green leaf volatiles; TAB, (E)-α-bergamotene; TBF, (E)-β-farnesene; —, not detected.",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "table-wrap"),
            ("position", 29),
            ("ordinal", 1),
            ("sibling_ordinal", 1),
            ("asset", None),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.030"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.030"),
            ("id", "SA1"),
            ("title", "Decision letter"),
            ("full_title", "Decision letter"),
            (
                "contributors",
                [
                    {
                        "type": "editor",
                        "role": "Reviewing editor",
                        "surname": "Greenberg",
                        "given-names": "Jean T",
                        "affiliations": [
                            {
                                "institution": "University of Chicago",
                                "country": "United States",
                            }
                        ],
                    }
                ],
            ),
            (
                "content",
                "eLife posts the editorial decision letter and author response on a selection of the published articles (subject to the approval of the authors). An edited version of the letter sent to the authors after peer review is shown, indicating the substantive concerns or comments; minor concerns are not usually shown. Reviewers have the opportunity to discuss the decision before the letter is sent (see review process). Similarly, the author response typically shows only responses to the major concerns raised by the reviewers. Thank you for sending your work entitled “Plant defense phenotypes determine the consequences of volatile emission for individuals and neighbors” for consideration at eLife. Your article has been favorably evaluated by Diethard Tautz (Senior editor), a Reviewing editor, and three reviewers. The Reviewing editor and the reviewers discussed their comments before we reached this decision, and the Reviewing editor has assembled the following comments to help you prepare a revised submission. This manuscript makes an important contribution to understanding the role of two volatiles. The reviewers identified a few issues that need improvement. 1) You report that the role of lox mediated direct defence and its associated volatiles is determining damage on the plot level, not affecting focal or neighbouring plants. Whereas the TSP10 plants do affect abundance of stem borer beetles and predator presence on individual plant level. This conclusion is sound based on the data presented, but it is also biased by the experimental design. You did not explore the role of LOX derived volatiles or direct defences on the individual plant level experimentally. You should acknowledge and discuss the fact that the effect of LOX mediated effects has not been tested fully. The Discussion should include how the missing treatment might affect your conclusions or further inferences about the role of particular volatile emissions. 2) Although from the plants perspective measurement of damage is a useful predictor of the costs of herbivory, the measurement may not accurately present the actual effect on herbivore abundance. Damage is confounding: herbivore abundance, their feeding intensity (that individuals may adjust to plant quality), and predator pressure on herbivores. Actual herbivore counts such as provided for the stem borer beetles would more strongly support the conclusions on LOX and TSP mediated community processes. Please discuss this point or provide the counts if you have them. 3) You rely very heavily on pairwise comparisons between treatments, usually with non-parametric tests or G tests for differences in proportions. The results would be much clearer to the readers, and avoid many of the concerns over multiple comparisons, if the authors used generalized linear models that better reflect the experimental design (see below for specific suggestion). For instance, the strongest results for their “community genetics” argument comes from the pattern of reduced mortality of lox2/3 vs. WT plants when in the presence of a TPS10 plant (Figure 9), but you do not test this directly. This pattern should be tested as an interaction between the lox and TPS treatments. You interpret this pattern strongly, but all of the tests performed show marginal or non-significant differences (a marginal decline in mortality for lox2/3 plants and a non-significant increase in mortality for WT plants due to the presence of a TPS10 plant). A similar model could be used for stem borer infestation and predation rates. In addition to providing more powerful tests, it may help the reader if all of these analyses are presented in a parallel way. Specific suggestion: your statistical analysis could be improved for the analysis of the field experiment data (mortality, prevalence of stem borers, and prevalence of predation). All of the outcome variables are binomial (1 or 0 at the level of an individual plant). It is recommended that you model this outcome with a generalized linear mixed model, using a binomial error distribution. This could be done with the glmer function in R or the GLIMMIX procedure in SAS, among other options. The model would include two crossed factors: the LOX genotype of the individual plant (2 levels, wild type or mutant) crossed with the TPS10 treatment of the population (5 levels, individual WT, individual TPS10, monoculture WT, monoculuture TPS10, mixed), and plot as a random effect to account for the non-independence of plants from the same plot. The test of whether the presence of a TPS10 plant alters mortality on non-TPS10 neighbors differently based on the LOX genotype of those neighbors would come from the interaction term in this model, with the TPS10 producing plant individuals excluded (and thus the TPS10 monoculture treatment dropped). Or alternatively, you could drop the individual plant treatments and have a three way factorial model of LOX genotype*population treatment (mono WT, mono TPS10, mixed)*position (Center vs. Edge). You could then perform an a priori contrast to compare the interaction specifically between the (WT monoculture vs. mixture)*(LOX genotype) for edge plants to test the significance of the pattern in Figure 9. 4) Please consider that a streamlining of the presentation would make the manuscript easier to read and understand. A great deal of space is devoted to establishing that the gene silencing/overexpression worked and produced reliable phenotypes in controlled conditions. These methods and results could be moved to an appendix. This would allow the authors to focus on the results from the field experiment, where the interesting finding is not just that manipulating plant defenses and volatiles affects herbivore damage and plant fitness, but that the genotype of one individual can affect the outcome on neighboring individuals. [Editors' note: further revisions were requested prior to acceptance, as described below.] Thank you for resubmitting your work entitled “Plant defense phenotypes determine the consequences of volatile emission for individuals and neighbors” for further consideration at eLife. Your revised article has been favorably evaluated by Diethard Tautz (Senior editor), a member of the Board of Reviewing Editors, and two of the original reviewers. The manuscript has been improved but there are some remaining issues that need to be addressed before acceptance, as outlined below. The reviewers found your manuscript to be improved. However there remains a statistical issue that requires resolution. This was framed by one of the reviewers as follows: ”My other concern with the original article was that the authors used a great deal of pairwise comparisons rather than building statistical models that better reflected their experimental design and could test directly for the interactive effects at the heart of their arguments. The authors made some attempt to respond to this suggestion, providing revamped analyses for plant size, flower number, and herbivore damage. However, I was disappointed to see that they did not collect their data in a way that would allow this more powerful analyses of the plant mortality data (which is the clearest case of “extended phenotypes” where the genotype of the center plant affected the fitness of the edge plants), or the data on stem borer abundance and predation rates. For the mortality data especially, this is surprising to me: why would the authors not have noted the live/dead status of plants individually, rather than just taking a tally? Do they mean that within a given population of five plants, they tallied the total number dead (1-5), or do they really mean that they cannot ascribe their mortality data even to the plot level (the base unit of replication for their experiment). If it is the former, they could still use a binomial generalized linear model, as long as they kept the data from the center and edge plants separately. If it is the latter, this is concerning to me because this analysis does not reflect the psuedoreplication induced by having five plants per plot. While a G test is “unreplicated” in that you need only enter a single frequency, it still assumes that the individuals comprising those frequencies are independent observations (i.e. a comparison between 28/75 vs. 50/75 is more powerful than 6/15 vs. 10/15). And I am truly puzzled as to how the authors can have data on flower production at an individual plant scale, but not have mortality data on that scale. Surely if they know how many flowers any given plant produced, they must know if it was alive or not. Am I missing something obvious here?” In light of these comments, please clarify how the data on plant mortality and stem borer infection was collected. If the mortality analysis is unreplicated as described in the cover letter, please justify why it is still appropriate to use the statistical test that you used.",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "sub-article"),
            ("position", 30),
            ("ordinal", 1),
            ("sibling_ordinal", 1),
            ("asset", "dec"),
        ]
    ),
    OrderedDict(
        [
            ("doi", "10.7554/eLife.04490.031"),
            ("doi_url", "https://doi.org/10.7554/eLife.04490.031"),
            ("id", "SA2"),
            ("title", "Author response"),
            ("full_title", "Author response"),
            (
                "content",
                "1) You report that the role of lox mediated direct defence and its associated volatiles is determining damage on the plot level, not affecting focal or neighbouring plants. Whereas the TSP10 plants do affect abundance of stem borer beetles and predator presence on individual plant level. This conclusion is sound based on the data presented, but it is also biased by the experimental design. You did not explore the role of LOX derived volatiles or direct defences on the individual plant level experimentally. You should acknowledge and discuss the fact that the effect of LOX mediated effects has not been tested fully. The Discussion should include how the missing treatment might affect your conclusions or further inferences about the role of particular volatile emissions. We agree that we did not always explicitly consider the limits of our experimental design when discussing the data, and to correct this, we have added the following text to the Discussion: a) “It should be noted that our experimental design did not support the analysis of population-level effects […] in the presence, or absence, of total HIPVs and jasmonate-mediated defense.” b) We changed the subsection title from “LOX2/3 expression has large direct effects on plants, while the largest effects of TPS10 expression are indirect” to “LOX2/3 expression has large direct effects on plants, while TPS10 expression has large indirect effects”. c) We have also added the following sentence: “In contrast, due to the large suite of defenses regulated by LOX2 and LOX3 products, it would have been more difficult to determine the proximate cause of neighbor effects resulting from manipulation of LOX2 and LOX3 expression.” d) We had already stated in the Discussion: “Although several components of plant HIPV blends have been shown to prime the direct and indirect defenses of neighbors in different systems (reviewed in Baldwin et al. 2006; Arimura et al. 2010; and see Heil, Kost 2006), GLVs may often be the active cues (Engelberth et al. 2004; Kessler et al. 2006; Frost et al. 2008). We did not mix GLV-deficient with GLV-emitting plants in our experimental design, but we can conclude that TPS10 volatiles had no detectable direct effects on neighbor plants in our set-up.” (This was in the first submission and has not changed.) e) “Of course it is possible that the manipulation of LOX2 and LOX3 expression within populations would also reveal such extortionist-like effects, for example if T. mucorea preferred to oviposit in TPS10 plants over LOX2/3xTPS10 plants in the same population, which would support the hypothesis that T. mucorea preference reflects the ratio between TPS10 volatiles and other HIPVs.” f) And finally, we added: “In this context, it would be interesting to know whether central LOX2/3 plants surrounded by WT edge plants would experience a similar predation disadvantage as central TPS10 plants surrounded by WT edge plants, and whether central WT plants surrounded by LOX2/3 edge plants would have an advantage over their neighbors in attracting predators.” 2) Although from the plants perspective measurement of damage is a useful predictor of the costs of herbivory, the measurement may not accurately present the actual effect on herbivore abundance. Damage is confounding: herbivore abundance, their feeding intensity (that individuals may adjust to plant quality), and predator pressure on herbivores. Actual herbivore counts such as provided for the stem borer beetles would more strongly support the conclusions on LOX and TSP mediated community processes. Please discuss this point or provide the counts if you have them. We thank the reviewers for this astute suggestion. We do indeed have one time point of herbivore count data and had previously not included it because the data are from only a subset of plants and we considered that they might not be as robust as the damage data. However, we re-visited these data and have now included them as a new Figure 6 (the original figures have been re-numbered accordingly). Indeed, the herbivore abundance data differs from the damage data. This must be interpreted with caution as the abundance data were recorded on June 26th while the damage data were recorded on June 9th, but we state these dates clearly in the text and note, in the Results and in the Discussion, that observation of the plants indicated that there were no changes in trends of herbivore damage between these dates (i.e., relative damage levels remained consistent throughout the end of the experiment). These data are also introduced in the Introduction, reviewed in the Results and in the Discussion, in several commented locations, and a new paragraph has been included in the Methods under the subsection headed “Herbivore and predator counts”. 3) You rely very heavily on pairwise comparisons between treatments, usually with non-parametric tests or G tests for differences in proportions. The results would be much clearer to the readers, and avoid many of the concerns over multiple comparisons, if the authors used generalized linear models that better reflect the experimental design (see below for specific suggestion). For instance, the strongest results for their “community genetics” argument comes from the pattern of reduced mortality of lox2/3 vs. WT plants when in the presence of a TPS10 plant (Figure 9), but you do not test this directly. This pattern should be tested as an interaction between the lox and TPS treatments. You interpret this pattern strongly, but all of the tests performed show marginal or non-significant differences (a marginal decline in mortality for lox2/3 plants and a non-significant increase in mortality for WT plants due to the presence of a TPS10 plant). A similar model could be used for stem borer infestation and predation rates. In addition to providing more powerful tests, it may help the reader if all of these analyses are presented in a parallel way. Specific suggestion.: your statistical analysis could be improved for the analysis of the field experiment data (mortality, prevalence of stem borers, and prevalence of predation). All of the outcome variables are binomial (1 or 0 at the level of an individual plant). It is recommended that you model this outcome with a generalized linear mixed model, using a binomial error distribution. This could be done with the glmer function in R or the GLIMMIX procedure in SAS, among other options. The model would include two crossed factors: the LOX genotype of the individual plant (2 levels, wild type or mutant) crossed with the TPS10 treatment of the population (5 levels, individual WT, individual TPS10, monoculture WT, monoculuture TPS10, mixed), and plot as a random effect to account for the non-independence of plants from the same plot. The test of whether the presence of a TPS10 plant alters mortality on non-TPS10 neighbors differently based on the LOX genotype of those neighbors would come from the interaction term in this model, with the TPS10 producing plant individuals excluded (and thus the TPS10 monoculture treatment dropped). Or alternatively, you could drop the individual plant treatments and have a three way factorial model of LOX genotype*population treatment (mono WT, mono TPS10, mixed)*position (Center vs. Edge). You could then perform an a priori contrast to compare the interaction specifically between the (WT monoculture vs. mixture)*(LOX genotype) for edge plants to test the significance of the pattern in Figure 9. We have revisited these raw data and regrettably, none of these data sets were collected in a manner which we think permits application of the modeling techniques suggested. Specifically, the Trichobaris mucorea infestation data and mortality data were simply collected as tallies per individual or population type and genotype at the end of the season, i.e., not replicated. Thus an unreplicated analysis such as the G-test, according to our understanding, must be used. The predation assays were not always conducted on exactly the same group of plants, as over the course of the 5 repetitions some plants were damaged by wind or severed by cutworms and new plants had to be assigned. Thus it would be difficult to appropriately apportion error to a random or repeated-measures term in a model, and we feel it is most appropriate for these data as well to treat them as total frequencies and conduct an unreplicated statistical analysis. We have stated these limitations clearly in the Methods and Results sections describing the relevant experiments, as marked by comments in the revised text. However, we greatly appreciate the reviewers’ points and detailed suggestions. We thus reviewed all of the data in the manuscript and determined that other data sets would be more appropriately analyzed with such an approach. We therefore used an approach similar to that suggested by the reviewers to analyze the newly included foliar herbivore abundance data, and we furthermore re-analyzed our foliar herbivore damage, plant size, and flower production data. These changed analyses are presented in the new version of Figure 5 (foliar herbivore damage data), the new Figure 6 and source data files (foliar herbivore abundance data), and a new Figure 8 and source data file (re-analyzed size and flower production data). The general statistical approaches are described in the revised statistical analysis section of the Materials and methods, and in addition, we have provided detailed descriptions of the analyses as part of the new Appendix 3. In particular, the re-analysis of plant size and reproduction data allowed us to simplify the presentation of the flowering data in Figure 7 to focus only on LOX2/3 deficiency effects, while revealing subtler interactive effects of TPS10 expression in the separate effects (new analysis presented in the new Figure 8 and Figure 8–Source data 1), which fit well with the other data sets, and which were overlooked in our previous analysis. Overall, the revised statistical analyses have revealed more population-level and interactive effects of TPS10 expression which we feel facilitate our discussion of the data, strengthen our conclusions, and overall provide a more comprehensible and coherent picture for the reader. 4) Please consider that a streamlining of the presentation would make the manuscript easier to read and understand. A great deal of space is devoted to establishing that the gene silencing/overexpression worked and produced reliable phenotypes in controlled conditions. These methods and results could be moved to an appendix. This would allow the authors to focus on the results from the field experiment, where the interesting finding is not just that manipulating plant defenses and volatiles affects herbivore damage and plant fitness, but that the genotype of one individual can affect the outcome on neighboring individuals. We thank the reviewers for this suggestion to streamline the text. The eLife staff suggested that we create three appendices, each with one coherent topic or set of contents. Thus, the results regarding screening of transgenic lines are now in Appendix 1, results demonstrating field-stable phenotypes are in Appendix 2, and the corresponding methods are in Appendix 3. We feel this has greatly improved the readability and organization of the manuscript. [Editors' note: further revisions were requested prior to acceptance, as described below.] The reviewers found your manuscript to be improved. However there remains a statistical issue that requires resolution. This was framed by one of the reviewers as follows: “My other concern with the original article was that the authors used a great deal of pairwise comparisons rather than building statistical models that better reflected their experimental design and could test directly for the interactive effects at the heart of their arguments. The authors made some attempt to respond to this suggestion, providing revamped analyses for plant size, flower number, and herbivore damage. However, I was disappointed to see that they did not collect their data in a way that would allow this more powerful analyses of the plant mortality data (which is the clearest case of “extended phenotypes” where the genotype of the center plant affected the fitness of the edge plants), or the data on stem borer abundance and predation rates. For the mortality data especially, this is surprising to me: why would the authors not have noted the live/dead status of plants individually, rather than just taking a tally? Do they mean that within a given population of five plants, they tallied the total number dead (1-5), or do they really mean that they cannot ascribe their mortality data even to the plot level (the base unit of replication for their experiment). If it is the former, they could still use a binomial generalized linear model, as long as they kept the data from the center and edge plants separately. If it is the latter, this is concerning to me because this analysis does not reflect the psuedoreplication induced by having five plants per plot. While a G test is “unreplicated” in that you need only enter a single frequency, it still assumes that the individuals comprising those frequencies are independent observations (i.e. a comparison between 28/75 vs. 50/75 is more powerful than 6/15 vs. 10/15). And I am truly puzzled as to how the authors can have data on flower production at an individual plant scale, but not have mortality data on that scale. Surely if they know how many flowers any given plant produced, they must know if it was alive or not. Am I missing something obvious here?” In light of these comments, please clarify how the data on plant mortality and stem borer infection was collected. If the mortality analysis is unreplicated as described in the cover letter, please justify why it is still appropriate to use the statistical test that you used. This is an important concern and a considered comment, and we want to give a correspondingly careful and reasoned response, which is below. In summary, the data presented in Figure 9 is the most biologically meaningful plant mortality data, which is unfortunately from an unreplicated data set; however the analysis presented in Figure 9 is appropriate for these data and our question, and does not violate the assumptions of the G-test regarding independence between the groups compared. Likewise, the assumption of independence is not violated for the analysis in Figure 11. However the assumption of independence of the Fisher’s test was violated in one instance for the analysis presented in Figure 10. We have corrected our analysis for Figure 10 and associated parts of the Results text, but the corrected analysis produces exactly the same conclusions. Question 1: Should we replace our unreplicated plant mortality data set (presented in Figure 9) with a replicated data set, collected concurrently with flowering data? Of course we did note plant mortality while collecting the data on flowering. However the flowering data was collected on May 29th and June 6th, while we wished to analyze mortality by the end of the season on June 29th. All other data sets collected between June 6th and June 29th (herbivore abundance, predation rates) were collected on subsets of plants and plant mortality was not noted again until June 29th, at which point all surviving plants were tallied in categories while checking stems for infestation by Trichobaris mucorea. Thus total mortality for the season was calculated on June 29th as the number of plants originally planted minus the number surviving in each category. Because these total mortality numbers from June 29th most accurately reflect the likelihood that plants would not have survived to produce ripe seed, we feel it is most meaningful to present the June 29th data. One could argue that, although the final mortality numbers from June 29th are principally more meaningful, nevertheless if mortality did not change much between June 6th and June 29th, it would be preferable to present a replicated rather than an unreplicated data set. However there was a 1.5-fold increase in overall mortality, from 29% to 46%, between June 6th and June 29th. Although the mortality data from June 6th show similar trends as June 29th, with LOX2/3 deficiency tending to increase mortality and TPS10 expression tending to lower mortality, the differences on June 6th are not significant even when analyzed using a GLM with a binomial distribution, because the total mortality is lower. Thus the June 6th dataset has lower statistical power and, more importantly, is truly less biologically meaningful. Answer to question 1: No. It is best to present the unreplicated June 29th data. Question 2: Is the use of G-tests to analyze the data in Figure 9 appropriate? What about the use of G-tests and Fisher’s tests in other data sets (Figures 10 and 11)? In analyzing total mortality on June 29th, we wished to know whether mortality rate was dependent on, or independent of, population type. We thus used G-tests of independence and conducted post-hoc P-value corrections for multiple comparisons as necessary, which we understand to be the most appropriate choice of tests for our unreplicated data and our question. Although our data came from experimental populations (plots), we did not violate the assumption of independence incorporated into the G-test for the data shown in Figure 9. This is because, in Figure 9, we only compare data from separate population types (separate plots), and we never compare data from plants at different positions in the same population type. If we had compared e.g. center to edge plants, there would be dependencies within the data set because some of the plants in group “edge” would have come from the same populations as plants in group “center”, and thus we would not be comparing independent observations. However by comparing different population types, we ensured that no plants in group A shared a population with any plants in group B, and therefore there are no dependencies between groups: they are independent. The same reasoning holds for the comparisons in Figure 11, and for the comparisons of mortality between plants differing only in LOX2/3 deficiency or from different population types made in the Results section text (in the subsection headed “LOX2/3 deficiency increases plant mortality, but mortality is mitigated by single TPS10-expressing plants in LOX2/3-deficient populations”). For Figure 10, we compare both plants from separate population types and plants from the same population type. The comparison of plants from the same population type violates the independence assumption of the Fisher’s test and we thank the Reviewer for bringing this to our attention. We have removed the incorrect comparisons, changed the representation of the statistical comparisons in Figure 10 accordingly, and corrected the relevant parts of the figure caption and of the manuscript (subsection headed “TPS10-expressing plants increase the infestation rates of their neighbors by the stem-boring weevil Trichobaris mucorea”). In this part of the manuscript text, we also removed the Fisher’s tests of infestation by genotype: when comparing genotypes differing only in TPS10 expression overall, rather than different population types, we also violated the assumption of independence because some populations contain both TPS10-expressing and non-expressing individuals. As a result of removing some of the pairwise comparisons, our post-hoc correction factor decreased and the P-value we report for the comparison of WT plants in mixed versus monocultures decreased from 0.025 to 0.015. Because the critical comparisons in the analysis of the Trichobaris mucorea infestation data are those made between, and not within populations, the correction of our statistics does not change the conclusions we draw from these data. Answer to Question 2: Yes, the G-tests and Fisher’s tests are appropriate and generally the requirement of independent observations is not violated. However, one comparison shown in Figure 10 and a few comparisons in the associated results text violated the independence assumption of the Fisher’s test. We have removed these comparisons. The corrected analysis does not change our conclusions at all.",
            ),
            ("article_doi", "10.7554/eLife.04490"),
            ("type", "sub-article"),
            ("position", 31),
            ("ordinal", 2),
            ("sibling_ordinal", 2),
            ("asset", "resp"),
        ]
    ),
]
